\pdfoutput = 1
\documentclass[12pt]{article}
%\documentclass[aps,prl,twocolumn]{revtex4-1}
%\documentclass[preprint]{revtex4-1}

\usepackage{amsmath}
\usepackage{amssymb}
\usepackage{graphicx}
\usepackage[margin =1 in]{geometry}
\usepackage{lineno}
\usepackage{hyperref}
%\usepackage{ulem}
%\usepackage{mathtools}
%\usepackage{microtype}
\usepackage{seqsplit}
\usepackage{courier}

%% Macros
\def\reals{\mathbb{R}}
\def\be{\begin{equation}}
\def\ee{\end{equation}}
\def\bea{\begin{eqnarray}}
\def\eea{\end{eqnarray}}
\def\bml{\begin{mathletters}}
\def\eml{\end{mathletters}}
\def\bse{\begin{subequations}}
\def\ese{\end{subequations}}
\def\expec{\mathbb{E}}
\def\exp{\text{exp}}
\def\Var{\text{Var}}
\def\e{\text{e}}
\def\ba{\begin{align}}
\def\ea{\end{align}}

\usepackage{titlesec}
%\titleformat*{\section}{\LARGE\bfseries\center}
%\titlespacing\section{0pt}{0pt}{0pt}
%\titleformat*{\subsection}{\Large\bfseries \raggedright}
%	\titlespacing\subsection{0pt}{0pt}{-6pt}
\titleformat{\subsubsection}[runin]{\itshape \raggedright}{}{}{}[]

%\titleformat*{\subsubsection}[runin]{\bfseries \raggedright}[]
%	\titlespacing\subsubsection{0pt}{0pt}{0pt}
%{left spacing}{before spacing}{afterspacing}


\begin{document}



\title{Supporting Information for: Sublethal antibiotics collapse gut bacterial populations by enhancing aggregation and expulsion}

%%%%% full author info %%%%%%%%
\author{Brandon H. Schlomann$^{1,2,}$*, Travis J. Wiles$^{1,}$*, Elena S. Wall$^{1}$, \\
Karen Guillemin$^{1,3}$, Raghuveer Parthasarathy$^{1,2}$}
\date{}

\maketitle
\setlength\parskip{12pt}
\setlength\parindent{0pt}


%%%%% Affiliations %%%%%%%%%%

$^1$Institute of Molecular Biology, University of Oregon, Eugene, Oregon, United States of America 

$^2$Department of Physics and Materials Science Institute, University of Oregon, Eugene, Oregon, United States of America 

$^3$Humans and the Microbiome Program, CIFAR, Toronto,  Ontario M5G 1Z8, Canada

*Equal contributors


\section*{Materials and Methods}

\subsection*{Animal care}
All experiments with zebrafish were done in accordance with protocols approved by the University of Oregon Institutional Animal Care and Use Committee and following standard protocols \cite{Westerfield2007}. 

\subsection*{Gnotobiology}
Wild-type (AB$\times$TU strain) zebrafish were derived germfree (GF) and colonized with bacterial strains as previously described \cite{Melancon2017} with slight modifications. Briefly, fertilized eggs from adult mating pairs were harvested and incubated in sterile embryo media (EM) containing ampicillin (100 $\mu$g/ml), gentamicin (10 $\mu$g/ml), amphotericin B (250 ng/ml), tetracycline (1 $\mu$g/ml), and chloramphenicol (1 $\mu$g/ml) for 6 hours. Embryos were then washed in EM containing 0.1\% polyvinylpyrrolidone-iodine followed by EM containing 0.003\% sodium hypochlorite. Sterilized embryos were distributed into T25 tissue culture flasks containing 15 ml sterile EM at a density of one embryo per milliliter and incubated at 28 to 30$^{\circ}$C prior to bacterial colonization. Embryos were sustained on yolk-derived nutrients and were not fed during experiments. For bacterial mono-association, bacteria were first grown overnight in lysogeny broth (LB) with shaking at 30$^{\circ}$C and were prepared for inoculation by pelleting 1 ml of culture for 2 min at 7,000$\times g$ and washing once in sterile EM. Bacterial strains were individually added to the water column of single flasks containing 4-day-old larval zebrafish at a final density of 10$^6$ bacteria/ml. For antibiotic treatment, drugs were added at the indicated working concentration directly to flask containing animals that had been colonized 24 hours prior. 

\subsection*{Bacterial strains and culture}
\textit{Vibrio cholerae} ZWU0020 and \textit{Enterobacter cloacae} ZOR0014 were originally isolated from the zebrafish intestine \cite{Stephens2016}. Fluorescently marked derivatives of each strain were previously generated by Tn\textit{7}-mediated insertion of a single constitutively expressed gene encoding dTomato \cite{Wiles2018}. We note that all plating- and imaging-based experiments performed in this study were done using fluorescently marked strains, which carry a gentamicin resistance cassette, with the exception of experiments in which fluorescent dyes were used to assess viability of cells. Archived stocks of bacteria were maintained in 25\% glycerol at -80$^{\circ}$C. Prior to experiments, bacteria were directly inoculated from frozen stocks into 5 ml LB media (10 g/L NaCl, 5 g/L yeast extract, 12 g/L tryptone, 1 g/L glucose) and grown for $\sim$16 hours (overnight) shaking at 30$^{\circ}$C.

\subsection*{Generation of a fluorescent SOS reporter}
To identify a suitable promoter within the \textit{Vibrio} ZWU0020 genome (\url{https://img.jgi.doe.gov/m/}, IMG genome ID: 2522572152) for creation of a genetically encoded fluorescent DNA-damage `SOS' reporter, we scanned the upstream regions of each gene for consensus gammaproteobacterial `SOS boxes' (\texttt{CTGTN$_8$ACAG}) that serve as binding sites for the repressor LexA (Fig. S7A and S7B) \cite{erill2007aeons}. Of the genes identified, the promoter of the gene \textit{recN} (IMG gene ID: 2705597027) was an ideal candidate for three main reasons: 1) it contains multiple SOS boxes (2 consensus and 2 with 2 mismatches), which is an arrangement that is potentially associated with tight/graded regulation \cite{kreuzer2013dna}; 2) the \textit{recN} promoter is highly conserved among closely related \textit{V. cholerae} strains as well as other non-\textit{Vibrio} gammaproteobacterial lineages, suggesting that \textit{recN} is a bona fide representative of the SOS response; and 3) \textit{recN} is one of the most highly expressed genes in response to DNA damaging agents in both \textit{E. coli} and \textit{V. cholerae} \cite{courcelle2001comparative,Krin2018}, likely due to its multiple near-consensus -10 promoter sequences.

We rationally designed a \textit{recN}-based fluorescent SOS reporter by fusing the 100bp \textit{recN} promoter region to an open reading frame (ORF) encoding superfolder green fluorescent protein (sfGFP) (Fig. S7C). In addition, we incorporated an epsilon enhancer and consensus Shine-Dalgarno sequence within the 5' untranslated region (UTR) to help ensure robust translation of the reporter gene \cite{Wiles2018,olins1989novel,olins1988t7}, and incorporated the synthetic transcriptional terminator L3S2P21 into the 3' UTR \cite{chen2013characterization}. We built the construct using polymerase chain reaction (PCR) and synthetic oligonucleotides. Primer WP97 (containing the \textit{recN} promoter and 5' UTR; 5'-\texttt{\seqsplit{TGAATGCATTAAAAGTGACCAAAAAATTTTACCTGAGTGACTTTACTGTATAAAGAAACAGTATAAACTGTTTAAACATACAGTATTGGTTAATCATACAGGTGCAAACTTAACTTTATCAAGGAGACTAAATCATGAGCAAGGGCGAGGAGCT}}-3') and primer WP98 (containing the 3' UTR; 5'-\texttt{\seqsplit{TGAACTAGTAAAACGAAAAAAGGCCCCCCTTTCGGGAGGCCTCTTTTCTGGAATTTTTATCACTTGTACAGCTCGTCCATG}}-3') were used to PCR-amplify sfGFP from the source plasmid pXS-sfGFP \cite{Wiles2018}. Engineered restriction sites flanking the amplicon (NsiI and SpeI) were then used to insert the construct into a variant of the Tn\textit{7} delivery vector pTn\textit{7}xKS, which also harbors a constitutively expressed \textit{dTomato} gene for tracking all bacterial cells (Fig. S7D) \cite{Wiles2018}. The resulting dual-reporter construct was then inserted into the ZWU0020 genome as previously described \cite{Wiles2018}. To verify reporter activity, disk diffusion assays were performed on agar plates with the genotoxic agent mitomycin C and, as a control, the cell wall-targeting beta-lactam antibiotic ampicillin (Fig. S7E). Mitomycin C induced robust expression of sfGFP whereas ampicillin did not.  


 

\subsection*{In vitro characterization of antibiotics}

\textit{Growth kinetics:} Growth kinetics of bacterial strains in vitro were measured using a FLUOstar Omega microplate reader. Prior to growth measurements, bacteria were grown overnight in 5 ml LB media at 30$^{\circ}$C with shaking. The next day, cultures were diluted 1:100 into fresh LB media with or without the indicated antibiotic and dispensed in quadruplicate (200 $\mu$l/well) into a sterile 96-well clear flat-bottom tissue culture-treated microplate. Absorbance at 600 nm was then recorded every 30 min for $\sim$16 hours at 30$^{\circ}$C with shaking. Growth rates were estimated by fitting a logistic growth curve to OD values, starting at manually defined points marking the end of lag phase.

\textit{Viability:} Cultures of \textit{Vibrio} ZWU0020 or \textit{Enterobacter} ZOR0014 were grown overnight in LB at 30$^{\circ}$C with shaking. The next day, 1:100 dilutions were made in fresh LB media containing either ciprofloxacin (\textit{Vibrio}: 10 ng/ml, \textit{Enterobacter}: 25 ng/ml) or no drug. Cultures were incubated at 30$^{\circ}$C with shaking for 6 hours prior to being stained using a LIVE/DEAD BacLight Bacterial Viability Kit according to manufacturer's instructions. Culture/stain mixtures were diluted 1:10 in 0.7\% saline and imaged using a Leica MZ10 F fluorescence stereomicroscope equipped with a 2.0X objective and a Leica DFC365 FX camera. Images were captured using standard Leica Application Suite software. Bacteria were identified in images with intensity-based region finding following difference of gaussians filtering. Cells stained in both SYTO9 and propidium iodide were identified as overlapping regions in the two color channels. Analysis code was written in MATLAB.

\textit{Cell length and swimming speed}: Dense overnight cultures of \textit{Vibrio} ZWU0020 were diluted 1:100 in fresh LB media alone or with 10 ng/ml ciprofloxacin and incubated at 30$^{\circ}$C with shaking for 4 h. Bacteria were then imaged on a Nikon TE2000 inverted fluorescence microscope between a slide and a coverslip using a 60X oil immersion objective and a Hamamatsu ORCA CCD camera (Hamamatsu City, Japan). Movies were taken within 60 seconds of mounting at an exposure time of 30 ms, resulting in a frame rate of 15 frames/sec, and had a duration of approximately 7 seconds. Bacteria in the resulting movies were identified with intensity-based region finding and tracked using nearest-neighbor linking. Analysis code was written in MATLAB. Five movies were taken per treatment case. For untreated length analysis, $n = 2291$ bacteria were quantified; for ciprofloxacin-treated length analysis, $n = 963$. For untreated speed analysis, $n = 833$ bacteria; for ciprofloxacin-treated speed analysis, $n = 531$.

\textit{Vibrio SOS reporter activity:} \textit{Vibrio} ZWU0020 carrying the fluorescent SOS reporter was grown overnight in LB at 30$^{\circ}$C with shaking. The next day, 1:100 dilutions were made in fresh LB media containing either 10 ng/ml ciprofloxacin, 400 ng/ml mitomycin C, 10 $\mu$g/ml ampicillin, or no drug. Cultures were then grown overnight ($\sim$16 h) at 30$^{\circ}$C with shaking. The next day, cultures were diluted 1:43 in 80\% glycerol (as an immobilizing agent) and imaged with a Nikon Eclipse Ti inverted microscope equipped with an Andor iXon3 888 camera using a 40x objective and 1.5x zoom. Bacteria were identified in images with gradient-based region finding, using a Sobel filter, following difference of gaussians filtering. Analysis code was written in MATLAB. As expected, the two DNA targeting drugs, ciprofloxcain and mitomycin C, induced the SOS response in subpopulations of cells, while the cell-wall targeting drug ampicillin did not. In computing SOS-positive fractions, filamented cells were counted as single cells.

\subsection*{Culture-based quantification of bacterial populations} 
Dissection of larval guts was done as described previously \cite{milligan2011study}. Dissected guts were harvested and placed in a 1.6 ml tube containing 500 $\mu$l sterile 0.7\% saline and $\sim$100 $\mu$l 0.5 mm zirconium oxide beads. Guts were then homogenized using a bullet blender tissue homogenizer for $\sim$25 seconds on power 4. Lysates were serially plated on tryptic soy agar (TSA) and incubated overnight at 30$^{\circ}$C prior to enumeration of CFU and determination of bacterial load. Typically an overnight incubation is sufficient to recover all viable cells; however, we note that ciprofloxacin treatment results in delayed colony growth on agar plates (likely due to growth arrest induced by DNA-damage). We empirically determined that, in the case of ciprofloxacin treatment, an incubation period 72 hours was required for complete resuscitation of viable cells on agar plates. For all culture-based quantification of bacterial populations in this study, the estimated limit of detection is 5 bacteria/gut and the limit of quantification is 100 bacteria/gut. Plating data plotted are pooled from a minimum of two independent experiments. Samples with zero countable colonies on the lowest dilution were set to the limit of detection prior to plotting and statistical analysis. Enumeration of flask water abundances by plating was performed identically to gut abundances, including the 72 hour incubation period.

\subsubsection*{Comparing antibiotic treatments between intestinal populations and flask water populations:}  To compare the effect of ciprofloxacin on populations in the intestine and in the flask water, we normalized treated abundances by the corresponding untreated median abundance (Fig. 2C and 3A). To account for variation in untreated bacterial dynamics between weekly batches of fish, we performed the normalization within each batch. Unnormalized data is available in the Supplemental Data File.


\subsection*{Light sheet fluorescence microscopy of live larval zebrafish}
\subsubsection*{Imaging intestinal bacteria:}  Live imaging of larval zebrafish was performed using a custom-built light sheet fluorescence microscope previously described in detail \cite{Jemielita2014}. Larvae are anesthetized with MS-222 (Tricane) and mounted into small glass capillaries containing 0.5\% agarose gel by means of a metal plunger. Larvae are then suspended vertically in an imaging chamber filled with embryo media and anesthetic and extruded out of the capillary such that the set agar plug sits in front of the imaging objective. The full intestine volume ($\sim$1200 $\times$ 300 $\times$ 150 microns) is imaged in four subregions that are registered in software after imaging. The imaging of a full intestine volume sampled at 1-micron steps between $z$-planes is imaged in $\sim$45 seconds. Excitation lasers at 488 and 561 nm wavelengths were tuned to a power of 5 mW prior to entering the imaging chamber. A 30 ms exposure time was used for all 3D scans and 2D movies. Time lapse imaging was performed overnight, except for the 3.5 hour imaging of \textit{Enterobacter} (Fig. 3C), which occurred during the day.

\subsubsection*{Viability staining of expelled aggregates:}  Germ-free larval zebrafish were colonized with wild type \textit{Vibrio} or \textit{Enterobacter} (without fluorescent markers) for 24 hours and then mounted into agarose plugs using small glass capillaries identically to the imaging procedure (above). Individual capillaries were suspended into isolated wells of a 24-well tissue culture plate filled with embryo media containing anesthetic or anesthetic + ciprofloxacin (10 ng/ml for \textit{Vibrio}, 25 ng/ml for \textit{Enterobacter}) and the larvae were extruded from the capillaries. Fish remained mounted for 24 hours, during which expelled bacteria remained caught in the agarose plug. After treatment, fish were pulled back into the capillaries and transferred to smaller wells of a 96 well plate containing embryo media, anesthetic, and the LIVE/DEAD BacLight Bacterial Viability stains SYTO9 and propridium iodide. Fish were stained according to kit instructions, with the exception of the incubation period being extended from 15 to 30 min to account for potential issues with the aggregate nature of the cells \cite{netuschil2014confusion}.  Following staining, fish were pulled again into the capillaries and transferred to the light sheet microscope for imaging. As shown in Figures S4 and S10, zebrafish cells stain in addition to bacterial cells, precluding accurate quantification of viable fractions.

\subsection*{Image analysis}

Bacteria were identified in three-dimensional light sheet fluorescence microscopy images using a custom MATLAB analysis pipeline previously described \cite{Jemielita2014,schlomann2018bacterial}, with minor changes. In brief, small objects (single cells and small aggregates) are identified using difference of Gaussians filtering. False positives are rejected with a combination of intensity thresholding (mostly noise) and manual removal (mostly host cells). Large aggregates are identified with a graph cut algorithm \cite{Boykov2004} that is seeded with either an intensity-based mask or a gradient-based mask. The average intensity of a single cell is estimated as the mean intensity of small objects, which is then used to estimate the number of cells contained in larger clusters by normalizing the total fluorescence intensity of each cluster. Spatial distributions along the length of the gut are computed using a manually drawn line drawn that defines the gut's center axis.


\subsection*{Kinetic model and stochastic simulations}
\subsubsection*{Choosing rate kernels:  }Our approach to choosing the size dependence of the rate parameters was to pick the simplest kernels consistent with key experimental observations. The first key observation, made in past work \cite{Jemielita2014,Wiles2016}, was that in between the expulsion of large aggregates population growth is well-described by a deterministic logistic function. Therefore, we chose a logistic growth kernel. The second key observation was that we occasionally encountered populations consisting of just a single, large aggregate and many single cells (Fig. S9E), which suggests that active fragmentation of single cells, most likely during growth phases, is the dominant fragmentation process. This notion is supported by time-lapse images of initial growth (Supplemental Movie 9) that depicts the creation of single cells during growth, in addition to the growth of three dimensional aggregates. Based on these observations, we made the assumption that single cell fragmentation is the sole fragmentation process, leading to what is known in other contexts as a ``chipping'' kernel \cite{krapivsky2010kinetic}. Beyond the chipping assumption, we had little evidence that informed how single cell fragmentation depends on the size of the aggregate, so we opted for the simplest choice of a constant, size-independent rate. Similarly for aggregation and expulsion, the size dependence of the rates is likely determined by complicated and uncharacterized fluid mechanical interactions of bacterial clusters in peristaltic-like flow, which we parsimoniously replace with a simple constant kernel for both processes. In aggregated populations, since it is only the loss of the largest clusters (of size $\mathcal{O}(K)$) that significantly impacts the system, we expect that it is the expulsion rate for these largest clusters that matters, rather than how the expulsion rate scales with cluster size. To test this notion, we ran simulations in which the expulsion rate scaled as a power of the cluster size, $n$, according to $\lambda(n) = \lambda(n/K)^{\nu}$, and varied the exponent $\nu$. This ansatz keeps the expulsion rate of clusters of size $K$ fixed for all values of $\nu$.  The result is that the cluster size distribution does not change within uncertainty values (Fig. S13), indicating that this approximation is valid.

For reference, we note that with these choices the model can be summarized by the following Smoluchowski equation, which describes the time evolution of the concentration of clusters of size $n$, $c_n$, in the thermodynamic limit of infinite system size:

\begin{align}
\dot{c}_n = \text{ }& \frac{\alpha}{2}\sum_{m=1}^{n} c_{n-m}c_m - \alpha c_n \sum_{m=1}^{\infty}c_m+\beta(c_{n+1}- c_n) + \beta\delta_{n,1}\sum_{m=1}^{\infty}c_m\nonumber\\[6pt]
&+r\left(1-\frac{\sum_{m=1}^{\infty}mc_m}{K}\right)\left[(n-1)c_{n-1} - nc_n\right]  - \lambda c_n. 
\end{align}

\noindent The four rate parameters are $\alpha$ (aggregation), $\beta$ (fragmentation), $r$ (growth), and $\lambda$ (expulsion), and $K$ is the carrying capacity. In the last term of the first line,  $\delta_{n,1}$ is the Kronecker delta with second argument equal to $1$. Of note, the first line of equation (1), containing just aggregation and fragmentation terms, was previously studied as a model of polymer chains and was shown to exhibit interesting non-equilibrium steady states and scaling behaviors that are due to the breaking of detailed balance by the chipping kernel \cite{krapivsky1996transitional}. In our system detailed balance is also broken, but for a different reason: our ``monomers''---single cells---are alive and self-replicating.

\textit{Simulations}: As each zebrafish intestine contains at most a few hundred bacterial clusters, finite size effects and stochasticity impact cluster statistics, so we implemented the model as a hybrid deterministic-stochastic simulation that follows the time evolution of individual clusters. Gillespie's direct method \cite{gillespie1977exact} was used to simulate stochastic aggregation, fragmentation, and expulsion events. Growth was treated as deterministic. Once the time until next stochastic reaction, $\tau$, was determined according to the Gillespie algorithm, integration was performed with the Euler method from time $t$ to $t+\tau$ using a time step $\Delta t = \text{min}(\tau, 0.1 \text{ hr})$. 

To simulate \textit{Vibrio} populations, direct stochastic simulation becomes intractable due to the large number of clusters ($\sim 10^5$ single cells). We therefore implemented a modified tau-leaping algorithm \cite{gillespie2001approximate} that facilitates large simulations. We opted for a straightforward fixed $\tau$ method and empirically determined that a value of $\tau = 0.001$ h produced no observable differences in cluster size and abundance distributions compared to direct stochastic simulation (SI Appendix, Fig. S12A,B).

All simulations were written in MATLAB and code is available at \url{https://github.com/bschloma/gac}.

\subsection*{Parameter inference}
The kinetic model presented in the main text has 5 parameters: rates of growth, expulsion, aggregation, and fragmentation, along with an overall carrying capacity. As discussed in the main text, we directly measured \textit{Enterobacter}'s growth rate and expulsion rate through time-lapse imaging. The uncertainty of the expulsion rate was estimated by the standard error, using the previously validated assumption that the expulsion of large aggregates follows a Poisson process \cite{Wiles2016}:

\be
\text{SE}_{\lambda} = \frac{\sqrt{\text{mean number of expulsions}}}{(\text{imaging time})\times\sqrt{\text{number of fish}}}.
\ee

For the remaining parameters, we developed a method to infer them from the distribution of abundances obtained from dissection and plating assays. In a regime where aggregation and fragmentation are fast compared to expulsion, we expect the system to locally reach a quasi-steady state in between expulsions of the largest aggregates. As such, we expect cluster statistics to depend primarily on the ratio of fragmentation to aggregation, $\beta/\alpha$, rather than on each rate independently. This confirmed in simulations (Fig. S11A and S11B). Therefore, the number of parameters to be estimated is reduced to two: $\beta/\alpha$ and $K$. 

\textit{Untreated Enterobacter}:  We fixed $\alpha = 0.1$ hr$^{-1}$ and performed a grid search in $\beta$ and $K$ on a logarithmic grid, simulating the model multiple trials for each pair of $(\beta,K)$. The number of trials decreased with increasing $\beta$, from 1000 to 10. Each simulation started from 10 single cells and ran for a simulated time of 64 hours, modeling our 72 hour colonization data with an 8 hour colonization window. To model static host-host variation, we drew each carrying capacity from a log-normal distribution with a standard deviation of 0.5 decades. This is the standard deviation of the untreated \textit{Vibrio} abundance distribution (Fig. S3F), which is an appropriate measure of static host-host variation because untreated \textit{Vibrio} does not form large aggregates and therefore does not experience large, stochastic population collapses due to aggregate expulsion. We then compared the mean ($\mu$) and variance ($\sigma$) of the simulated, log-transformed abundances $\log_{10}(N+1)$ with the values for our plating data ($\hat{\mu}$ and $\hat{\sigma}$, respectively), quantifying error using 

\be
\chi^2 = (\mu - \hat{\mu})^2 + (\sigma - \hat{\sigma})^2.
\ee

A heat map of $\chi^2$ shows well-defined edges for the minimum values of the fit parameters (Fig. S11C). However, the inference is poorly constrained for carrying capacities larger than $10^5$ and for $\log_{10} \beta/\alpha$  greater than 2.5. This poor constraint is due primarily to the insensitivity of the abundance distribution to increasing values of these parameters. For example, moving to the far right side of the abundance phase diagram in Fig. 5B, the contours become flat in $\beta/\alpha$. 

To further constrain our estimates, we place upper bounds on these parameters with simple estimates of physical limits. To bound the carrying capacity, we note that a larval zebrafish intestine have a volume of roughly 1 nl, or $10^6$ $\mu$m$^3$. Taking the volume of a bacterium to be roughly 1 $\mu$m$^3$, we estimate a maximum bacterial load of 10$^6$ cells, consistent with the largest \textit{Vibrio} abundances (Fig. S3F). As we find no \textit{Enterobacter} populations above 10$^{5.5}$, and in our simulations we draw carrying capacities from a log-normal distribution with a standard deviation of half a decade, we constrained our best fit estimate to $\log_{10}K = 5.0$. To bound the fragmentation rate, $\beta$, we considered the time-lapse movie that showcases the greatest degree of cluster fragmentation observed (Supplemental Movie 9). This movie depicts the initial growth phase, in which both the size of aggregates and the number of single cells increase. Because the aggregates visibly grow in size, we know that the fragmentation rate must be bounded by the absolute growth rate of the population, $\beta < rN$; if the fragmentation rate were larger, cells would break off of the aggregate faster than they would be produced by cell division, and the aggregates would shrink in size. Taking, roughly, $r\sim 10^{-1}$ and $N\sim 10^3$ (Fig. 4D), we estimate that $\beta < 10^2$, or, with $\alpha = 10^{-1}$, $\beta/\alpha < 10^3$. With this bound, we constrain our best fit estimate to $\log_{10}\beta/\alpha = 2.5$. We took the uncertainties of the best fit estimates, $\sigma_{\log_{10}K}$ and $\sigma_{\log_{10}\beta/\alpha}$, to be the inverse of the local curvatures of $\chi^2$ at the best fit values: $\sigma_{\theta} = 1/|\partial^2_{\theta}\chi^2|$, for $\theta = \log_{10}K$, $\log_{10}\beta/\alpha$, resulting in $\sigma_{\log_{10}K} = 0.5$ and $\sigma_{\log_{10}K} = 0.4$.
 
\textit{Ciprofloxacin-treated Enterobacter}:  To estimate the change in \textit{Enterobacter}'s parameters upon antibiotic treatment, we conservatively assumed equal effects on growth and fragmentation/aggregation and modeled treatment parameters as $r' = \epsilon r$ and $\beta' = \epsilon \beta$. We then performed a single parameter grid search of $\epsilon$ values, ranging from $10^{-1.75}$ to $10^{-0.5}$. We modeled the antibiotic treatment as a parameter quench with a 6 hour buffer time, in which the antibiotics entered the intestine and began to take action on the bacteria. The value of 6 hours was chosen based on the \textit{Vibrio} time series data. Each simulation was initialized with a cluster configuration drawn randomly from the imaging-derived dataset of actual untreated \textit{Enterobacter} populations. The parameters $r$, $\lambda$, and $K$ were set to their best fit or measured values, $\alpha$ was again fixed at 0.1 hr$^{-1}$, and  $r$ and $\beta$ were both scaled by the same factors of $\epsilon$. We then ran simulations for a modified simulation time $24 - 6 = 18$ hours and fit the mean and standard deviation of shifted log-transformed abundances measured in the 24 hour treatment plating assays. A plot of $\chi^2$ vs $\epsilon$ shows a clear minimum at $\epsilon = 10^{-1}$ (Fig. S11D).

\textit{Untreated Vibrio}: Untreated \textit{Vibrio} populations are comprised of almost entirely single cells and therefore represent an extreme limit of the kinetic model. In this regime, fragmentation is so thorough that even dividing cells immediately separate and there is no appreciable aggregation. Because multicellular clusters are extremely rare, our data are insufficient to extract numerical estimates of model parameters. However, one can estimate a lower bound for the fragmentation rate, $\beta$, by equating it to the total growth rate, $rN$, where $N$ is the total population size; i.e. clusters do not grow without fragmenting. This estimate yields $\beta \gtrsim 10^5$. For the expulsion rate, if we assume the same rate as \textit{Enterobacter} (positing unchanged intestinal mechanics), we obtain $r/\lambda \sim 7$ These values place untreated \textit{Vibrio} off-scale in the phase diagram of Fig. 5B.
 
\textit{Ciprofloxacin-treated Vibrio}: We performed a two-parameter fit to ($\beta/\alpha$, $r$), using the measured expulsion rate for \textit{Enterobacter} ($\lambda = 0.11$ h$^{-1}$ and the typical untreated \textit{Vibrio} abundance for a carrying capacity of $K \sim 10^5$. We observed that in approaching the extinction transition from above, simulated abundance distributions transition from unimodal to bimodal in shape, with a peak emerging near $N=0$ representing populations that suffered large, abrupt collapses. As such, fitting just the mean and variance as was done for \textit{Enterobacter} produced inaccurate estimates. Therefore, we implemented full maximum likelihood estimation using 100 simulated replicates to estimate the likelihood. While the fit to treated \textit{Vibrio} resulted in less-constrained parameter estimates in the $r-\beta$ plane compared to the Enterobacter fit, it did yield a clear maximum (Fig. S12C) and a best-fit abundance distribution that matched experimental data within uncertainties (Fig. S12D). Like with \textit{Enterobacter}, we can attempt to assess the validity of this model by comparing the now-parameter-free prediction of the cluster size distribution with the image-derived data. Due to the rarity of large clusters and to limited data, the experimental distribution is severely undersampled. It shows, however, qualitative agreement with the model prediction (Fig. S12E). Finally, to confirm that our choice of the simulation timestep $\tau$ did not affect our parameter estimation, we decreased $\tau$ by a factor of 2 from $0.001$ h to $0.0005$ h and found no change in the best-fit cluster size distribution within sampling uncertainties (Fig. S12F). Because our parameter grid used in the fit was coarse, we estimate the uncertainty of our best-fit parameters as the grid spacing. Our uncertainty values are therefore likely overestimated.




  

\newpage
\bibliography{/Users/brandonschlomann/Documents/Gutz/PaperWriting/abx/abx_refs.bib}{}
\bibliographystyle{unsrt}
%\bibliographystyle{/Users/brandonschlomann/Documents/Gutz/PaperWriting/abx/science/Science.bst}

%\begin{thebibliography}{10}
%
%\bibitem{dethlefsen2011incomplete}
%Les Dethlefsen and David~A Relman.
%\newblock Incomplete recovery and individualized responses of the human distal
%  gut microbiota to repeated antibiotic perturbation.
%\newblock {\em Proceedings of the National Academy of Sciences}, 108(Supplement
%  1):4554--4561, 2011.
%
%\bibitem{cho2012antibiotics}
%Ilseung Cho, Shingo Yamanishi, Laura Cox, Barbara~A Meth{\'e}, Jiri Zavadil,
%  Kelvin Li, Zhan Gao, Douglas Mahana, Kartik Raju, Isabel Teitler, et~al.
%\newblock Antibiotics in early life alter the murine colonic microbiome and
%  adiposity.
%\newblock {\em Nature}, 488(7413):621, 2012.
%
%\bibitem{schulfer2019impact}
%Anjelique~F. Schulfer, Jonas Schluter, Yilong Zhang, Quincy Brown, Wimal
%  Pathmasiri, Susan McRitchie, Susan Sumner, Huilin Li, Joao~B. Xavier, and
%  Martin~J. Blaser.
%\newblock The impact of early-life sub-therapeutic antibiotic treatment
%  ({STAT}) on excessive weight is robust despite transfer of intestinal
%  microbes.
%\newblock {\em The ISME Journal}, 2019, doi:10.1038/s41396-019-0349-4,
%  published online ahead of print.
%
%\bibitem{gaulke2016triclosan}
%Christopher~A Gaulke, Carrie~L Barton, Sarah Proffitt, Robert~L Tanguay, and
%  Thomas~J Sharpton.
%\newblock Triclosan exposure is associated with rapid restructuring of the
%  microbiome in adult zebrafish.
%\newblock {\em PLoS One}, 11(5):e0154632, 2016.
%
%\bibitem{andersson2014microbiological}
%Dan~I Andersson and Diarmaid Hughes.
%\newblock Microbiological effects of sublethal levels of antibiotics.
%\newblock {\em Nature Reviews Microbiology}, 12(7):465, 2014.
%
%\bibitem{national2018environmental}
%Engineering "National Academies~of Sciences, Medicine, and others".
%\newblock {\em Environmental Chemicals, the Human Microbiome, and Health Risk:
%  A Research Strategy}.
%\newblock National Academies Press, 2018.
%
%\bibitem{walters2003contributions}
%Marshall~C Walters, Frank Roe, Amandine Bugnicourt, Michael~J Franklin, and
%  Philip~S Stewart.
%\newblock Contributions of antibiotic penetration, oxygen limitation, and low
%  metabolic activity to tolerance of pseudomonas aeruginosa biofilms to
%  ciprofloxacin and tobramycin.
%\newblock {\em Antimicrobial Agents and Chemotherapy}, 47(1):317--323, 2003.
%
%\bibitem{fux2005survival}
%CA~Fux, J~William Costerton, Philip~S Stewart, and Paul Stoodley.
%\newblock Survival strategies of infectious biofilms.
%\newblock {\em Trends in Microbiology}, 13(1):34--40, 2005.
%
%\bibitem{korem2015growth}
%Tal Korem, David Zeevi, Jotham Suez, Adina Weinberger, Tali Avnit-Sagi, Maya
%  Pompan-Lotan, Elad Matot, Ghil Jona, Alon Harmelin, Nadav Cohen, et~al.
%\newblock Growth dynamics of gut microbiota in health and disease inferred from
%  single metagenomic samples.
%\newblock {\em Science}, 349(6252):1101--1106, 2015.
%
%\bibitem{schlomann2018bacterial}
%Brandon~H Schlomann, Travis~J Wiles, Elena~S Wall, Karen Guillemin, and
%  Raghuveer Parthasarathy.
%\newblock Bacterial cohesion predicts spatial distribution in the larval
%  zebrafish intestine.
%\newblock {\em Biophysical Journal}, 115(11):2271--2277, 2018.
%
%\bibitem{Moor2017}
%Kathrin Moor, M{\'{e}}d{\'{e}}ric Diard, Mikael~E. Sellin, Boas Felmy,
%  Sandra~Y. Wotzka, Albulena Toska, Erik Bakkeren, Markus Arnoldini, Florence
%  Bansept, Alma~Dal Co, Tom V{\"{o}}ller, Andrea Minola, Blanca
%  Fernandez-Rodriguez, Gloria Agatic, Sonia Barbieri, Luca Piccoli, Costanza
%  Casiraghi, Davide Corti, Antonio Lanzavecchia, Roland~R. Regoes, Claude
%  Loverdo, Roman Stocker, Douglas~R. Brumley, Wolf~Dietrich Hardt, and Emma
%  Slack.
%\newblock {High-avidity IgA protects the intestine by enchaining growing
%  bacteria}.
%\newblock {\em Nature}, 544(7651):498, 2017.
%
%\bibitem{welch2017spatial}
%Jessica L~Mark Welch, Yuko Hasegawa, Nathan~P McNulty, Jeffrey~I Gordon, and
%  Gary~G Borisy.
%\newblock Spatial organization of a model 15-member human gut microbiota
%  established in gnotobiotic mice.
%\newblock {\em Proceedings of the National Academy of Sciences},
%  114(43):E9105--E9114, 2017.
%
%\bibitem{cremer2017effect}
%Jonas Cremer, Markus Arnoldini, and Terence Hwa.
%\newblock Effect of water flow and chemical environment on microbiota growth
%  and composition in the human colon.
%\newblock {\em Proceedings of the National Academy of Sciences},
%  114(25):6438--6443, 2017.
%
%\bibitem{Stephens2016}
%W.~Zac Stephens, Adam~R. Burns, Keaton Stagaman, Sandi Wong, John~F. Rawls,
%  Karen Guillemin, and Brendan~J.M. Bohannan.
%\newblock {The composition of the zebrafish intestinal microbial community
%  varies across development}.
%\newblock {\em ISME Journal}, 10:644--654, 2016.
%
%\bibitem{Wiles2016}
%Travis~J. Wiles, Matthew Jemielita, Ryan~P. Baker, Brandon~H. Schlomann,
%  Savannah~L. Logan, Julia Ganz, Ellie Melancon, Judith~S. Eisen, Karen
%  Guillemin, and Raghuveer Parthasarathy.
%\newblock {Host Gut Motility Promotes Competitive Exclusion within a Model
%  Intestinal Microbiota}.
%\newblock {\em PLoS Biology}, 14(7):1--24, 2016.
%
%\bibitem{Wiles2018}
%Travis~J. Wiles, Elena~S. Wall, Brandon~H. Schlomann, Edouard~A. Hay, Raghuveer
%  Parthasarathy, and Karen Guillemin.
%\newblock Modernized tools for streamlined genetic manipulation and comparative
%  study of wild and diverse proteobacterial lineages.
%\newblock {\em mBio}, 9(5):e01877--18, 2018.
%
%\bibitem{relmanABX_2011}
%Les Dethlefsen and David~A. Relman.
%\newblock Incomplete recovery and individualized responses of the human distal
%  gut microbiota to repeated antibiotic perturbation.
%\newblock {\em Proceedings of the National Academy of Science}, 108, 2011.
%
%\bibitem{girardi2011biodegradation}
%Cristobal Girardi, Josephine Greve, Marc Lamsh{\"o}ft, Ingo Fetzer, Anja
%  Miltner, Andreas Sch{\"a}ffer, and Matthias K{\"a}stner.
%\newblock Biodegradation of ciprofloxacin in water and soil and its effects on
%  the microbial communities.
%\newblock {\em Journal of Hazardous Materials}, 198:22--30, 2011.
%
%\bibitem{goneau2015subinhibitory}
%Lee~W Goneau, Thomas~J Hannan, Roderick~A MacPhee, Drew~J Schwartz, Jean~M
%  Macklaim, Gregory~B Gloor, Hassan Razvi, Gregor Reid, Scott~J Hultgren, and
%  Jeremy~P Burton.
%\newblock Subinhibitory antibiotic therapy alters recurrent urinary tract
%  infection pathogenesis through modulation of bacterial virulence and host
%  immunity.
%\newblock {\em mBio}, 6(2):e00356--15, 2015.
%
%\bibitem{rennekamp201515}
%Andrew~J Rennekamp and Randall~T Peterson.
%\newblock 15 years of zebrafish chemical screening.
%\newblock {\em Current opinion in chemical biology}, 24:58--70, 2015.
%
%\bibitem{yoganantharjah2017use}
%Prusothman Yoganantharjah and Yann Gibert.
%\newblock The use of the zebrafish model to aid in drug discovery and target
%  validation.
%\newblock {\em Current topics in medicinal chemistry}, 17(18):2041--2055, 2017.
%
%\bibitem{erill2007aeons}
%Ivan Erill, Susana Campoy, and Jordi Barb{\'e}.
%\newblock Aeons of distress: an evolutionary perspective on the bacterial sos
%  response.
%\newblock {\em FEMS Microbiology Reviews}, 31(6):637--656, 2007.
%
%\bibitem{kreuzer2013dna}
%Kenneth~N Kreuzer.
%\newblock Dna damage responses in prokaryotes: regulating gene expression,
%  modulating growth patterns, and manipulating replication forks.
%\newblock {\em Cold Spring Harbor Perspectives in Biology}, 5(11):a012674,
%  2013.
%
%\bibitem{irazoki2016sos}
%Oihane Irazoki, Albert Mayola, Susana Campoy, and Jordi Barb{\'e}.
%\newblock Sos system induction inhibits the assembly of chemoreceptor signaling
%  clusters in salmonella enterica.
%\newblock {\em PLoS One}, 11(1):e0146685, 2016.
%
%\bibitem{Logan2018}
%Savannah~L. Logan, Jacob Thomas, Jinyuan Yan, Ryan~P. Baker, Drew~S. Shields,
%  Joao~B. Xavier, Brian~K. Hammer, and Raghuveer Parthasarathy.
%\newblock {The Vibrio cholerae type VI secretion system can modulate host
%  intestinal mechanics to displace gut bacterial symbionts}.
%\newblock {\em Proceedings of the National Academy of Sciences},
%  115(16):E3779--E3787, 2018.
%
%\bibitem{ganz2018}
%J.~Ganz, R.~P. Baker, M.~K. Hamilton, E.~Melancon, P.~Diba, J.~S. Eisen, and
%  R.~Parthasarathy.
%\newblock Image velocimetry and spectral analysis enable quantitative
%  characterization of larval zebrafish gut motility.
%\newblock {\em Neurogastroenterology \& Motility}, 30(9):e13351, 2018.
%
%\bibitem{robinson2018experimental}
%Catherine~D Robinson, Helena~S Klein, Kyleah~D Murphy, Raghuveer Parthasarathy,
%  Karen Guillemin, and Brendan~JM Bohannan.
%\newblock Experimental bacterial adaptation to the zebrafish gut reveals a
%  primary role for immigration.
%\newblock {\em PLoS biology}, 16(12):e2006893, 2018.
%
%\bibitem{krapivsky2010kinetic}
%Pavel~L Krapivsky, Sidney Redner, and Eli Ben-Naim.
%\newblock {\em A Kinetic View of Statistical Physics}.
%\newblock Cambridge University Press, 2010.
%
%\bibitem{krapivsky1996transitional}
%PL~Krapivsky and S~Redner.
%\newblock Transitional aggregation kinetics in dry and damp environments.
%\newblock {\em Physical Review E}, 54(4):3553, 1996.
%
%\bibitem{bansept2019enchained}
%Florence Bansept, Kathrin Moor-Schumann, Mederic Diard, Wolf-Dietrich Hardt,
%  Emma~Wetter Slack, and Claude Loverdo.
%\newblock Enchained growth and cluster dislocation: a possible mechanism for
%  microbiota homeostasis.
%\newblock {\em bioRxiv}, doi: 10.1101/298059, 2019.
%
%\bibitem{baharoglu2011vibrio}
%Zeynep Baharoglu and Didier Mazel.
%\newblock Vibrio cholerae triggers sos and mutagenesis in response to a wide
%  range of antibiotics: a route towards multiresistance.
%\newblock {\em Antimicrobial Agents and Chemotherapy}, 55(5):2438--2441, 2011.
%
%\bibitem{guerin2009sos}
%{\'E}milie Guerin, Guillaume Cambray, Neus Sanchez-Alberola, Susana Campoy,
%  Ivan Erill, Sandra Da~Re, Bruno Gonzalez-Zorn, Jordi Barb{\'e},
%  Marie-C{\'e}cile Ploy, and Didier Mazel.
%\newblock The sos response controls integron recombination.
%\newblock {\em Science}, 324(5930):1034--1034, 2009.
%
%\bibitem{beaber2004sos}
%John~W Beaber, Bianca Hochhut, and Matthew~K Waldor.
%\newblock Sos response promotes horizontal dissemination of antibiotic
%  resistance genes.
%\newblock {\em Nature}, 427(6969):72, 2004.
%
%\bibitem{tropini2017gut}
%Carolina Tropini, Kristen~A Earle, Kerwyn~Casey Huang, and Justin~L Sonnenburg.
%\newblock The gut microbiome: connecting spatial organization to function.
%\newblock {\em Cell Host \& Microbe}, 21(4):433--442, 2017.
%
%\bibitem{Westerfield2007}
%M.~Westerfield.
%\newblock {The Zebrafish Book. A Guide for the Laboratory Use of Zebrafish
%  (Danio rerio), 5th Edition}.
%\newblock {\em University of Oregon Press, Eugene (Book)}, 2007.
%
%\bibitem{Melancon2017}
%E.~Melancon, S.~{Gomez De La Torre Canny}, S.~Sichel, M.~Kelly, T.~J. Wiles,
%  J.~F. Rawls, J.~S. Eisen, and K.~Guillemin.
%\newblock {Best practices for germ-free derivation and gnotobiotic zebrafish
%  husbandry}.
%\newblock {\em Methods in Cell Biology}, 138:61--100, 2017.
%
%\bibitem{courcelle2001comparative}
%Justin Courcelle, Arkady Khodursky, Brian Peter, Patrick~O Brown, and Philip~C
%  Hanawalt.
%\newblock Comparative gene expression profiles following uv exposure in
%  wild-type and sos-deficient escherichia coli.
%\newblock {\em Genetics}, 158(1):41--64, 2001.
%
%\bibitem{Krin2018}
%Evelyne Krin, Sebastian~Aguilar Pierl{\'e}, Odile Sismeiro, Bernd Jagla,
%  Marie-Agn{\`e}s Dillies, Hugo Varet, Oihane Irazoki, Susana Campoy, Zo{\'e}
%  Rouy, St{\'e}phane Cruveiller, Claudine M{\'e}digue, Jean-Yves Copp{\'e}e,
%  and Didier Mazel.
%\newblock Expansion of the sos regulon of vibrio cholerae through extensive
%  transcriptome analysis and experimental validation.
%\newblock {\em BMC Genomics}, 19(1):373, May 2018.
%
%\bibitem{olins1989novel}
%Peter~O Olins and SH~Rangwala.
%\newblock A novel sequence element derived from bacteriophage t7 mrna acts as
%  an enhancer of translation of the lacz gene in escherichia coli.
%\newblock {\em Journal of Biological Chemistry}, 264(29):16973--16976, 1989.
%
%\bibitem{olins1988t7}
%Peter~O Olins, Catherine~S Devine, Shaukat~H Rangwala, and Kamilla~S Kavka.
%\newblock The t7 phage gene 10 leader rna, a ribosome-binding site that
%  dramatically enhances the expression of foreign genes in escherichia coli.
%\newblock {\em Gene}, 73(1):227--235, 1988.
%
%\bibitem{chen2013characterization}
%Ying-Ja Chen, Peng Liu, Alec~AK Nielsen, Jennifer~AN Brophy, Kevin Clancy, Todd
%  Peterson, and Christopher~A Voigt.
%\newblock Characterization of 582 natural and synthetic terminators and
%  quantification of their design constraints.
%\newblock {\em Nature Methods}, 10(7):659, 2013.
%
%\bibitem{milligan2011study}
%Kathryn Milligan-Myhre, Jeremy~R Charette, Ryan~T Phennicie, W~Zac Stephens,
%  John~F Rawls, Karen Guillemin, and Carol~H Kim.
%\newblock Study of host--microbe interactions in zebrafish.
%\newblock In {\em Methods in Cell Biology}, volume 105, pages 87--116.
%  Elsevier, 2011.
%
%\bibitem{Jemielita2014}
%Matthew Jemielita, Michael~J Taormina, Adam~R Burns, Jennifer~S Hampton,
%  Annah~S Rolig, Karen Guillemin, and Raghuveer Parthasarathy.
%\newblock Spatial and temporal features of the growth of a bacterial species
%  colonizing the zebrafish gut.
%\newblock {\em mBio}, 5(6):1--8, 2014.
%
%\bibitem{netuschil2014confusion}
%Lutz Netuschil, Thorsten~M Auschill, Anton Sculean, and Nicole~B Arweiler.
%\newblock Confusion over live/dead stainings for the detection of vital
%  microorganisms in oral biofilms-which stain is suitable?
%\newblock {\em BMC Oral Health}, 14(1):2, 2014.
%
%\bibitem{Boykov2004}
%Y.~Boykov and V.~Kolmogorov.
%\newblock {An experimental comparison of min-cut/max- flow algorithms for
%  energy minimization in vision}.
%\newblock {\em IEEE Transactions on Pattern Analysis and Machine Intelligence},
%  26(9):1124--1137, 2004.
%
%\bibitem{gillespie1977exact}
%Daniel~T Gillespie.
%\newblock Exact stochastic simulation of coupled chemical reactions.
%\newblock {\em The Journal of Physical Chemistry}, 81(25):2340--2361, 1977.
%
%\bibitem{gillespie2001approximate}
%Daniel~T Gillespie.
%\newblock Approximate accelerated stochastic simulation of chemically reacting
%  systems.
%\newblock {\em The Journal of Chemical Physics}, 115(4):1716--1733, 2001.
%
%\bibitem{ben2005genetic}
%Kaouther Ben-Amor, Hans Heilig, Hauke Smidt, Elaine~E Vaughan, Tjakko Abee, and
%  Willem~M de~Vos.
%\newblock Genetic diversity of viable, injured, and dead fecal bacteria
%  assessed by fluorescence-activated cell sorting and 16s rrna gene analysis.
%\newblock {\em Applied and Environmental Microbiology}, 71(8):4679--4689, 2005.
%
%\end{thebibliography}
%





%%%% sup fig captions

\newpage
\section*{Figure S1 caption}
\textbf{Measurement of \textit{Enterobacter} growth rate.} Image-derived quantification of initial growth dynamics in three zebrafish hosts. Imaging began approximately 8 hours after inoculation. 

\section*{Figure S2 caption}
\textbf{In vitro characterization of \textit{Vibrio} response to ciprofloxacin}. A: In vitro growth curves of \textit{Vibrio} in rich media (lysogeny broth) with different ciprofloxacin concentrations. B-C: Effects of ciprofloxacin on \textit{Vibrio} cell length and speed, with grey indicating experiments without antibiotic treatment and blue indicating exposure to 10 ng/ml ciprofloxacin. B: Distribution of \textit{Vibrio} cell lengths. Insets show representative fluorescence microscopy images of untreated and 10 ng/ml ciprofloxacin-treated cells; inset heights = 3.5 $\mu$m. C: Distribution of in vitro swimming speeds of individual bacteria.

\section*{Figure S3 caption}
\textbf{Additional \textit{Vibrio} data}  A: Representative masks of fluorescence microscopy images of in vitro viability staining. Top row, untreated, bottom row, 10 ng/ml ciprofloxacin-treated cells (6 hour treatment). SYTO9, shown in green (left panel), indicates intact cells, propridium iodine (PI), shown in magenta (middle panel), indicates dead cells. Double positive cells indicate damaged but viable cells \cite{ben2005genetic}, shown in white in the merged, right panel. Scale bar = 100 $\mu$m. B: Quantification of in vitro viability staining by fraction of cells corresponding to each case. Mean and standard deviation across 2 replicates shown. C: Representative fluorescence microscopy images of the SOS response in untreated (top row) and 10 ng/ml ciprofloxacin treated (bottom) cells. Constitutive dTom expression is shown in magenta (left), \textit{recN}-linked GFP expression in green (middle), merged images shown in right panel. Scale bar = 50 $\mu$m. D: Quantification of SOS response in fraction of SOS+ cells (Materials and Methods), mean and standard deviations shown, $n>4$ per treatment, total number of bacteria $>$ 120 cells per treatment. E: Timeline of in vivo antibiotic treatment. F: In vivo abundances of untreated and 10 ng/ml ciprofloxacin-treated cohorts by day. Each small circle corresponds to a single host, black lines indicate medians and quartiles. 

\section*{Figure S4 caption}
\textbf{Viability staining of \textit{Vibrio} cells expelled from the gut shows that ciprofloxacin does not induce widespread bacterial death in vivo}. Three examples of fish stained with SYTO9, which indicates live bacteria, and propidium iodide (PI), which indicates dead bacteria, for both untreated (A) and 10 ng/ml ciprofloxacin-treated (B) \textit{Vibrio}. Images were obtained by light sheet fluorescence microscopy and are maximum intensity projections of 3D images stacks. The field of view is around the vent region, as shown in the fish schematic at the top of the figure. The approximate boundary of the fish is indicated by the dashed orange line. Zebrafish cells also stain and constitute the bulk of the fluorescence in the images. Examples of zebrafish cells are indicated by white arrow heads. Examples of bacterial cells are indicated by the cyan arrows.


\section*{Figure S5 caption}
\textbf{In vivo ciprofloxacin dose response for \textit{Vibrio}:} \textit{Vibrio} was mono-associated with germ-free larval zebrafish for 24 hours prior to being left untreated, or treated with either 1, 10, or 100 ng/ml ciprofloxacin for an additional 24 hours.  \textit{Vibrio} abundances were determined by dissection and plating. Each circle corresponds to a single host intestine, black lines indicate medians and quartiles. Data for the `untreated' and `cipro 10 ng/ml' groups were included in Figure 2D and Supplemental Figure 2F, where they were combined with repeated experiments. 

\section*{Figure S6 caption}
\textbf{\textit{Vibrio} does not form large aggregates in vitro in response to ciprofloxacin}. Representative fluorescence microscopy images of untreated (A) and 10 ng/ml ciprofloxacin-treated (B) \textit{Vibrio} cells. Sample preparation and treatment are described in the \textit{Cell length and swimming speed} portion of the Materials and Methods section.

\section*{Figure S7 caption}
\textbf{Design and construction of fluorescent SOS reporter} A: Alignment of 100bp \textit{recN} promoter region plus start codon for the closely related \textit{V. cholerae} strains ZWU0020 (zebrafish isolate used in this study, IMG gene ID: 2705597027, locus tag: ZWU0020\_01601), ZOR0036 (zebrafish isolate, IMG gene ID: 2705599600, locus tag: ZOR0036\_00266), and El Tor N16961 (human pandemic isolate, IMG gene ID: 637047325, locus tag: VC0852). B: Alignment of 100bp \textit{recN} promoter region plus start codon of the \textit{V. cholerae} consensus \textit{recN} promoter, \textit{Aeromonas veronii} (zebrafish isolate, IMG gene ID: 2526373590, locus tag: L972\_03073), and \textit{E. coli} HS (human commensal isolate IMG gene ID: 640921890, locus tag: EcHS\_A2774). For panels A and B, SOS boxes are shaded based on their conservation to the consensus gammaproteobacterial sequence (\texttt{CTGTN$_8$ACAG}); `ATG' start codons are bolded; putative ribosome binding sites are boxed; and putative, near-consensus -10 promoter sequences (\texttt{TATAAT}) are bolded and underlined. C: Schematic of \textit{recN}-based fluorescent SOS reporter. Promoter comprises the consensus \textit{V. cholerae} \textit{recN} promoter region (P\textit{recN}), which was derived from the sequence alignment in panel A. The synthetic 5' untranslated region (UTR) contains an epsilon enhancer and consensus Shine-Dalgarno sequence. The open reading frame (ORF) encodes superfolder green fluorescent protein (sfGFP). And the 3' UTR contains the synthetic transcriptional terminator L3S2P21. D: Schematic of assembled SOS reporter in the context of the Tn\textit{7} tagging construct. Tn\textit{7}L and Tn\textit{7}R inverted repeats flank the Tn\textit{7} transposon. The SOS reporter was inserted upstream of a dTomato gene that is constitutively expressed from a synthetic Ptac promoter. A gene encoding gentamicin resistance (\textit{gentR}) was used to facilitate genetic manipulation. E: Disk diffusion assays verifying SOS reporter activity. \textit{Vibrio} ZWU0020 carrying the SOS reporter was spread onto agar plates using glass beads at a density that would give rise to a lawn of growth. Circular disks of Whatman filter paper (amber dashed lines) loaded with either the genotoxic agent mitomycin C or the cell wall-targeting beta-lactam antibiotic ampicillin were then placed in the center of the agar plates. After overnight incubation at 30$^{\circ}$C, plates were imaged using a fluorescence stereomicroscope. In the presence of mitomycin C, cells adjacent to the zone of inhibition (i.e., the area where there is no bacterial growth) robustly expressed sfGFP whereas in the presence of ampicillin they did not.    


\section*{Figure S8 caption}
In vitro growth curves (in lysogeny broth) of \textit{Enterobacter} with varying concentrations of ciprofloxacin.

\section*{Figure S9 caption}
\textbf{Additional \textit{Enterobacter} data}  A: Representative fluorescence microscopy images of in vitro viability staining. Top row, untreated, bottom row, 25 ng/ml ciprofloxacin-treated cells (6 hour treatment). SYTO9, shown in green (left panel), indicates intact cells, propridium iodine (PI), shown in magenta (middle panel), indicates dead cells. Double positive cells indicate damaged but viable cells \cite{ben2005genetic}, shown in white in the merged, right panel. Scale bar = 100 $\mu$m. B: Quantification of in vitro viability staining by fraction of cells corresponding to each case. Mean and standard deviation across 2 replicates shown. C: Timeline of in vivo antibiotic treatment. D: In vivo abundances of untreated and 25 ng/ml ciprofloxacin-treated cohorts by day. Each small circle corresponds to a single host, black lines indicate medians and quartiles. E: Maximum intensity projection of untreated \textit{Enterobacter} population showing an example of a population containing a single large cluster and several single cells. Scale bar = 200 $\mu$m.

\section*{Figure S10 caption}
\textbf{Viability staining of \textit{Enterobacter} cells expelled from the gut shows that ciprofloxacin does not induce widespread bacterial death in vivo}. Three examples of fish stained with SYTO9, which indicates live bacteria, and propidium iodide (PI), which indicates dead bacteria, for both untreated (A) and 25 ng/ml ciprofloxacin-treated (B) \textit{Enterobacter}. Images were obtained by light sheet fluorescence microscopy and are maximum intensity projections of 3D images stacks. The field of view is around the vent region, as shown in the fish schematic at the top of the figure. The approximate boundary of the fish is indicated by the dashed orange line. Zebrafish cells also stain and constitute the bulk of the fluorescence in the images. Examples of zebrafish cells are indicated by white arrow heads. Examples of bacterial cells are indicated by the cyan arrows.

\section*{Figure S11 caption}
\textbf{Additional model details}  A-B: Simulated heatmap of mean (A) and standard deviation (B) of $\log_{10}(\text{abundance} + 1)$ for varying values of aggregation and fragmentation rates. Both mean and standard deviation depend primarily on the ratio of fragmentation to aggregation rates, rather than on each rate independently. Dashed magenta line in (A) represents $\alpha = \beta$. Parameters: $r = 0.27$ hr$^{-1}$, $\lambda = 0.11$ hr$^{-1}$, $K = 10^5$, simulation time $= 64$ hours, number of trials decreased logarithmically with $\beta$ from 1000 to 10. Units of $\alpha$ and $\beta$ are hr$^{-1}$.  C: Heatmap of $\chi^2$ for untreated \textit{Enterobacter} fit to 7 dpf abundances (Materials and Methods). Parameters: $r = 0.27$ hr$^{-1}$, $\lambda = 0.11$ hr$^{-1}$, $\alpha = 0.1$ hr$^{-1}$, simulation time $= 64$ hours, number of trials decreased logarithmically with $\beta$ from 1000 to 10. D:   $\chi^2$ for fit to 6 dpf ciprofloxacin-treated \textit{Enterobacter} abundances as a function of the scaling parameter $\epsilon$, which scales the growth and fragmentation rates simultaneously according to $r \to \epsilon r$ and $\beta \to \epsilon \beta$. A clear minimum is seen at $\epsilon = 0.1$. Parameters: $r = 0.27$ hr$^{-1}$, $\lambda = 0.11$ hr$^{-1}$, $\alpha = 0.1$ hr$^{-1}$, $\beta = 10^{1.5}$ hr$^{-1}$, simulation time $= 64$ hours, number of trials decreased logarithmically with $\beta$ from 1000 to 10. E: 3D phase diagram of $\log_{10}(\text{abundance} + 1)$ with axes fragmentation/aggregation ($\beta/\alpha$), growth rate ($r$), and expulsion rate ($\lambda$). Blue isosurface represents $\log_{10}(\text{abundance} + 1) = 0.5 \pm 0.5$, yellow isosurface represents  $\log_{10}(\text{abundance} + 1) = 5.5 \pm 0.5$. Parameters: $\alpha = 0.1$ hr$^{-1}$, simulation time $= 64$ hours, number of trials decreased logarithmically with $\beta$ from 1000 to 10. Units of $\alpha$ and $\beta$ are hr$^{-1}$. F: Slices through the 3D phase diagram in (E) for different values of $\lambda$.

\section*{Figure S12 caption}
\textbf{Tau leaping simulations and \textit{Vibrio} parameter inference}. A-B: Comparison of direct stochastic simulation (``ssa'', gray circles) and our fixed-tau leaping (``tau'', purple diamonds) algorithm with $\tau = 0.001$ h. Simulations using both methods were run with the best-fit parameters for untreated \textit{Enterobacter} and 100 replicates. Both the cluster size distribution (A) and abundance histogram (B) show excellent agreement between the two methods. C-E: Details of model fit to ciprofloxacin-treated \textit{Vibrio} 24 h abundances. C: Heat map of log-likelihood. A manual grid search was performed over growth rate ($r$) and fragmentation rate ($\beta$). D: Comparison of the best-fit abundance distribution (purple line) to experimental data (blue circles). E: Comparison of the predicted cluster size distribution (purple line) to experimental data (blue circles). Here, all model parameters were fixed at their previously determined, best-fit values; there were no additional free parameters. The experimental data distribution is severely undersampled, estimated from just 4 fish. F: Confirmation that the best-fit solution is independent of our choice of $\tau$, indicating that simulations were performed with sufficient resolution. Simulations were run with the best-fit parameters but with $\tau$ decreased by a factor of 2, from $\tau=0.001$ h (purple circles) to $\tau = 0.0005$ h (green diamonds). Distributions agree with one another within sampling uncertainties.

\section*{Figure S13 caption}
\textbf{Model cluster size distributions are independent of how expulsion rate scales with cluster size}. Simulations were run with the expulsion rate depending on cluster size according to $E_n = \lambda (n/K)^{\nu}$, with $K$ the carrying capacity, and the exponent $\nu$ was varied. This ansatz keeps the expulsion rate of clusters of size $K$ constant. The resulting cluster size distributions agree with one another within sampling uncertainties, which are smaller than the marker size. This result justifies our use of the simple constant form of the expulsion kernel, $E_n = \lambda$.

\newpage
%\renewcommand\thefigure{\arabic{figure}}    
\renewcommand{\figurename}{Supplementary Figure }


\setcounter{figure}{0}

\section*{Supplementary Figure 1}
\begin{figure*}[h]
\centerline{
	\includegraphics[width =3 in]{figS1.eps}}
	%\caption{}
\end{figure*}



\newpage
\section*{Supplementary Figure 2}

\begin{figure*}[h!]
\centerline{
	\includegraphics[width =6 in]{figS2.eps}}
	
\end{figure*}

\newpage
\section*{Supplementary Figure 3}

\begin{figure*}[h!]
\centerline{
	\includegraphics[width =6 in]{figS3.eps}}
	
\end{figure*}


\newpage
\section*{Supplementary Figure 4}

\begin{figure*}[h!]
\centerline{
	\includegraphics[width =4.5 in]{figS4.eps}}
	
\end{figure*}

\newpage
\section*{Supplementary Figure 5}

\begin{figure*}[h!]
\centerline{
	\includegraphics[width =3 in]{figS5.eps}}
	
\end{figure*}

\newpage
\section*{Supplementary Figure 6}

\begin{figure*}[h!]
\centerline{
	\includegraphics[width =6 in]{figS6.eps}} 
	
\end{figure*}

\newpage
\section*{Supplementary Figure 7}

\begin{figure*}[h!]
\centerline{
	\includegraphics[width =6 in]{figS7.eps}} 
	
\end{figure*}

\newpage
\section*{Supplementary Figure 8}

\begin{figure*}[h!]
\centerline{
	\includegraphics[width =3 in]{figS8.eps}} 
	
\end{figure*}

\newpage
\section*{Supplementary Figure 9}

\begin{figure*}[h!]
\centerline{
	\includegraphics[width =6 in]{figS9.eps}} 
	
\end{figure*}

\newpage
\section*{Supplementary Figure 10}

\begin{figure*}[h!]
\centerline{
	\includegraphics[width =4.5 in]{figS10.eps}} 
	
\end{figure*}

\newpage
\section*{Supplementary Figure 11}

\begin{figure*}[h!]
\centerline{
	\includegraphics[width =6 in]{figS11.eps}} 
	
\end{figure*}

\newpage
\section*{Supplementary Figure 12}

\begin{figure*}[h!]
\centerline{
	\includegraphics[width =6 in]{figS12.eps}} 
	
\end{figure*}

\newpage
\section*{Supplementary Figure 13}

\begin{figure*}[h!]
\centerline{
	\includegraphics[width =3 in]{figS13.eps}} 
	
\end{figure*}
\newpage

\section*{Supplemental Movie captions}

\subsection*{Supplemental Movie 1} 
Light sheet fluorescence microscopy movie of untreated \textit{Vibrio} swimming in a 6 dpf zebrafish gut. The density of cells is highest on the left (anterior), where single cells cannot be resolved and the population appears as a single bright region (see also Figure 1C). On the right (posterior), single cells are more easily resolved and are seen swimming in and out of the intestinal folds. Each frame is from the same optical plane. Scale bar = 50 $\mu$m.

\subsection*{Supplemental Movie 2} 
Animated z-stack of light sheet fluorescence microscopy images of untreated \textit{Enterobacter} in a 6 dpf zebrafish gut. Bacterial clusters (bright white puncta) of diverse sizes are evident, from single cells up to a single cluster containing thousands of cells that appears at a z depth of $\sim$ 70 $\mu$m. Hazy reflection of light off of the fish's swim bladder can be seen outside the intestinal boundary in the upper right section of the images. Scale bar = 50 $\mu$m.

\subsection*{Supplemental Movie 3}
Fluorescence microscopy movie of untreated \textit{Vibrio} swimming between a glass slide and a coverslip (Materials and Methods). Scale bar = 20 $\mu$m.

\subsection*{Supplemental Movie 4}
Fluorescence microscopy movie of \textit{Vibrio} treated with 10 ng/ml ciprofloxacin swimming between a glass slide and a coverslip (Materials and Methods). Cells have undergone filamentation. Scale bar = 20 $\mu$m.

\subsection*{Supplemental Movie 5}
Time-lapse light sheet fluorescence microscopy movie of an established \textit{Vibrio} population responding to 10 ng/ml ciprofloxacin. Each frame is a maximum intensity projection of the full 3D intestinal volume.  The time between frames is 20 min. Initially, the population consists of a dense collection of individual, motile cells (Supplemental Movie 1, Figure 1C). Antibiotics are added after the second frame of the movie.  Following motility loss, cells leave the swarm and are compacted into aggregates, which are subject to strong transport down the length of the intestine and are eventually expelled. Scale bar = 200 $\mu$m.

\subsection*{Supplemental Movie 6}
Light sheet fluorescence microscopy movies of \textit{Vibrio} in fish treated with 10 ng/ml ciprofloxacin. The left panel movie shows constitutive dTom expression. The right panel movie was taken immediately after the left panel movie and shows a GFP reporter of the SOS response (Materials and Methods), which is expressed in cells strongly affected by ciprofloxacin (Fig. S3C and S3D). GFP-positive cells swim slowly or are aggregated. Each frame is from the same optical plane. Scale bar = 25 $\mu$m. 

\subsection*{Supplemental Movie 7}
Time-lapse light sheet fluorescence microscopy movie of an untreated \textit{Enterobacter} population showing an example of the expulsion process. Each frame is a maximum intensity projection of the full 3D intestinal volume. Time between frames is 10 min. The population is initially comprised of many small bacterial clusters and a single large cluster. Over time, small clusters are incorporated into the large one and the mass is transported down the length of the gut and expelled. Image intensities are log-transformed. Scale bar = 200 $\mu$m.

\subsection*{Supplemental Movie 8}
Time-lapse light sheet fluorescence microscopy movie of an untreated \textit{Enterobacter} population showing an example of the aggregation process. Each frame is a maximum intensity projection of the full 3D intestinal volume. Time between frames is 10 min. A collection of initially disconnected bacterial clusters on the left (anterior) side of the field of view gradually combine into a single cluster. Image intensities are log-transformed. Scale bar = 200 $\mu$m.

\subsection*{Supplemental Movie 9}
Time-lapse light sheet fluorescence microscopy movie of an untreated \textit{Enterobacter} population showing examples of the growth and fragmentation processes. Each frame is a maximum intensity projection of the full 3D intestinal volume. The time between frames is 20 min. The movie begins 8 hours after the initial exposure to \textit{Enterobacter}, by which time a small founding population has been established. Over time, the aggregates grow in size as cells divide, and new single cells also appear in the vicinity of the aggregates, likely due to fragmentation. Individual cell divisions from planktonic cells are also visible. Image intensities are log-transformed. Scale bar = 200 $\mu$m.

\subsection*{Supplemental Movie 10}
Light sheet fluorescence microscopy movie of \textit{Vibrio} in a fish treated with 10 ng/ml ciprofloxacin for $\sim$18 hours. Each frame is from the same optical plane, which spans the anterior-most region of the intestine known as the intestinal bulb (Fig. 1B). The bright signal in the left (anterior) side of the frame is a dense, motile swarm of planktonic cells (Supplemental Movie 1 and Fig. 1C). Moving from left to right (anterior-posterior) across the field of view, cells exhibiting filamentation and reduced motility are evident, along with the beginnings of small aggregates. Scale bar = 50 $\mu$m.

\subsection*{Supplemental Movie 11}
Light sheet fluorescence microsocopy movie of \textit{Vibrio} in a fish treated with 10 ng/ml ciprofloxacin for $\sim$18 hours. Each frame from the same single optical plane that captures a portion of the midgut (Fig. 1B). The bright signal is an aggregate of \textit{Vibrio} cells that nearly fills the width of the midgut lumen. Two cells are seen swimming near the end of the movie. Scale bar = 25 $\mu$m.





\end{document}