% chapter4.tex
% abx

%% Macros
\def\reals{\mathbb{R}}
\def\be{\begin{equation}}
\def\ee{\end{equation}}
\def\bea{\begin{eqnarray}}
\def\eea{\end{eqnarray}}
\def\bml{\begin{mathletters}}
\def\eml{\end{mathletters}}
\def\bse{\begin{subequations}}
\def\ese{\end{subequations}}
\def\expec{\mathbb{E}}
\def\exp{\text{exp}}
\def\Var{\text{Var}}
\def\e{\text{e}}
\def\ba{\begin{align}}
\def\ea{\end{align}}

\chapter{Sublethal antibiotics collapse gut bacterial populations by enhancing aggregation and expulsion}

\section{Statement of author contributions}
This chapter was published as \cite{schlomann_sublethal_2019}, a co-authored publication. Co-authors included Travis Wiles, a co-first author along with myself, Elena Wall, Karen Guillemin, and Raghuveer Parthasarathy. My contributions included: designing the study, performing the imaging, analyzing the data, developing the model, performing simulations, and writing the paper. 

\section{Abstract}
Antibiotics induce large and highly variable changes in the intestinal microbiome even at sublethal concentrations, through mechanisms that remain elusive. Using gnotobiotic zebrafish, which allow high-resolution examination of microbial dynamics, we found that sublethal doses of the common antibiotic ciprofloxacin cause severe drops in bacterial abundance. Contrary to conventional views of antimicrobial tolerance, disruption was more pronounced for slow-growing, aggregated bacteria than for fast-growing, planktonic species. Live imaging revealed that antibiotic treatment promoted bacterial aggregation and increased susceptibility to intestinal expulsion. Intestinal mechanics therefore amplify the effects of antibiotics on resident bacteria. Microbial dynamics are captured by a biophysical model that connects antibiotic-induced collapses to gelation phase transitions in soft materials, providing a framework for predicting the impact of antibiotics on the intestinal microbiome.












 \section{Introduction}


    Antibiotic drugs induce large, long-lasting, and disease-associated alterations in the composition of the intestinal microbiota \cite{relmanABX_2011,cho2012antibiotics,schulfer2019impact}. Even at concentrations well below the minimum inhibitory levels of many bacteria, antibiotics can lead to major and highly variable changes in the gut microbiome through mechanisms that remain mysterious \cite{cho2012antibiotics,schulfer2019impact,gaulke2016triclosan}. Sublethal antibiotics can also significantly alter animal physiology; the intentional growth enhancement of livestock is a well-known example that may involve microbiome-mediated pathways \cite{cho2012antibiotics}. Low concentrations of antibiotics are often present in the environment as byproducts of unchecked agricultural and biomedical use, generating public health concerns associated with the emergence of drug resistance \cite{andersson2014microbiological} as well as more direct impacts on human health \cite{national2018environmental}. It is therefore crucial to uncover mechanisms by which sublethal antibiotics reshape resident gut microbial communities. Understanding why particular bacterial strains are resilient or susceptible to antibiotic perturbations may allow us to predict the consequences of environmental contamination and may enable tailoring of antibiotic treatments as a therapeutic tool for manipulating the intestinal microbiome.  

\begin{figure}%[h]
\centerline{
	%\includegraphics[width =4.75 in]{abx/fig1.eps}}
	\includegraphics[width =4.75 in]{abx/fig1-eps-converted-to.pdf}}
	\caption{\textbf{Two bacterial species show different extremes of in vivo aggregation phenotypes.}}{  A: Schematic of a zebrafish 5 days post-fertilization (dpf). B: Schematic of the larval zebrafish intestine with numbers denoting approximate fraction of gut length. C: \textit{Vibrio cholerae} ZWU0020 in vivo. Left: a maximum intensity projection of a three-dimensional image of the full gut. Dense, bright bacteria and dimmer intestinal autofluorescence are evident. The orange dashed curve indicates a coarse outline of the gut boundary. Scale bar: 200 $\mu$m. Right: a single optical plane within the anterior bulb in a fish colonized with 1:100 green fluorescent protein (GFP): dTomato (dTom)-expressing \textit{Vibrio}, with the GFP channel shown to highlight individual microbes in the dense swarm. The orange dashed curve indicates the approximate contour of the intestinal epithelium. Black arrowheads indicate examples of single planktonic cells. Scale bar: 25 $\mu$m. (See also SI Movie 1) D: \textit{Enterobacter cloacae} ZOR0014 in vivo, shown as a maximum intensity projection of the full gut (left) and a subset of the same projection in the anterior bulb (right); bacterial aggregates are evident. The black arrowhead indicates an example of a single planktonic cell; the white arrowhead indicates an example of a multicellular aggregate. Scale bars same as in (C).}
\end{figure}


Conventional wisdom regarding bacterial responses to antibiotic drugs, derived largely from in vitro assays, holds that drug tolerance is facilitated by low growth rates and biofilm formation \cite{walters2003contributions,fux2005survival}. Recent work suggests that microbes in the vertebrate gastrointestinal tract adopt a variety of growth and aggregation phenotypes \cite{korem2015growth,schlomann_bacterial_2018,moor_high-avidity_2017,welch_spatial_2017}, raising the question of whether antibiotic susceptibility in the gut bears the same relationship to kinetics and physical structure as in less dynamic environments, or whether the strong mechanical activity and large fluid flows present in the intestine \cite{cremer2017effect} lead to fundamentally different rules.

To investigate the in vivo response of gut bacteria to low-dose antibiotic exposure, especially the relationship between susceptibility and bacterial behavior, we conducted live imaging-based studies of larval zebrafish (Fig. 1A, 1B), spanning the entire intestinal volume with spatial and temporal resolutions not attainable in humans or other model vertebrates. We focused our study on two native zebrafish bacterial isolates, both frequently found in the intestine \cite{stephens_composition_2016}, that we identified as representing extremes of growth and aggregation phenotypes \cite{schlomann_bacterial_2018}. The first, \textit{Vibrio cholerae} ZWU0020, hereafter referred to as ``\textit{Vibrio}'', exists in the larval zebrafish intestine primarily as dense populations of highly motile and planktonic individuals (Fig. 1C, SI Movie 1).  \textit{Vibrio} grows rapidly, with an in vivo doubling time of approximately 1 hour (exponential growth rate of 0.8 $\pm$ 0.3 1/hr) \cite{wiles_host_2016}. The second,  \textit{Enterobacter cloacae} ZOR0014, hereafter referred to as ``\textit{Enterobacter}'' primarily forms large, dense bacterial aggregates with small sub-populations of non-motile planktonic cells (Fig. 1D, SI Movie 2) \cite{wiles_modernized_2018} and has an in vivo doubling time of approximately 2.5 hours (exponential growth rate of 0.27 $\pm$ 0.05 1/hr) (SI Appendix, Fig. S1). To delineate and quantify antibiotic responses independent of inter-bacterial competition, we studied \textit{Vibrio} and \textit{Enterobacter} separately in hosts that were initially raised germ-free (Materials and Methods). We assessed response dynamics of each bacterial population after treatment with the antibiotic ciprofloxacin, a broad spectrum fluoroquinolone that interferes with DNA replication by inhibiting DNA gyrase. Ciprofloxacin is widely administered therapeutically and has been used as a model antibiotic in studies of human microbiome disruption  \cite{relmanABX_2011}. Furthermore, ciprofloxacin is often detected in environmental samples at ng/ml concentrations that are sublethal but capable of perturbing bacterial physiology \cite{girardi2011biodegradation, goneau2015subinhibitory}. 


As detailed below, we discovered that sublethal levels of ciprofloxacin lead to major reductions in intestinal abundance of both \textit{Vibrio} and \textit{Enterobacter} that could not be predicted from in vitro responses alone. In contrast to conventional wisdom, the slow-growing and highly aggregated \textit{Enterobacter} was impacted far more severely than the fast-growing, planktonic \textit{Vibrio}. Changes in bacterial abundances were driven primarily by clearance from the intestine by peristaltic-like fluid flow, which impacts aggregated bacteria more severely than planktonic cells. Exposure to sublethal levels of ciprofloxacin shifted both species to a more aggregated state, but for \textit{Enterobacter} this state was unsustainable and led to population collapse and extinction. Quantitative image-derived population data motivate and are well fit by physical models originally used to describe colloidal growth and polymer gelation, implying an antibiotic-induced phase transition in bacterial community physical structure and revealing a general framework for understanding and predicting intestinal antibiotic perturbations.









 \section{Results}

 \subsection{Low-dose ciprofloxacin increases bacterial aggregation and intestinal expulsion}

For both \textit{Vibrio} and \textit{Enterobacter}, we empirically determined a ciprofloxacin dosage that induced clear changes in bacterial physiology and behavior in vitro, but that was below the apparent minimum inhibitory concentration. We first describe results of antibiotic exposure, in vitro and in vivo, for the \textit{Vibrio} species. 



\begin{figure}%[H!]
	\centerline{
		%\includegraphics[width =4.75 in]{../../dissertation_figs/abx/fig2.eps}}
		\includegraphics[width =4.75 in]{abx/fig2-eps-converted-to.pdf}}
		\caption{\textbf{Low-dose ciprofloxacin induces \textit{Vibrio} aggregation and expulsion in vivo.}}{(Continued in footnote.)}
		\label{fig:fig2}
\end{figure}


{\let\thefootnote\relax\footnote{FIGURE \ref{fig:fig2} (cont'd) A: Schematic of the experimental timeline. B: Schematic of the sampling scheme for plating measurements. C: Normalized abundances (number of colony forming units (CFUs) scaled by untreated medians) of water and gut populations. $N$ values left to right: 8, 24, 7, 20, 6, 18, 5, 20. Water $N$ values denote number of flasks; gut $N$ values denote number of fish. D: Histograms of gut CFUs with pooled data from 24 and 48 h treatments. Counts indicate the number of individual fish with a given log$_{10}$ \textit{Vibrio} CFUs. Dashed lines indicate the mean of each set, showing a $\sim$100-fold reduction in intestinal \textit{Vibrio} abundance in antibiotic-treated fish. E: Ensemble-averaged spatial distributions of log-transformed cell density as a function of distance along the gut axis, integrated over the perpendicular dimensions. F: Maximum intensity projections of 3D images of untreated (top) and ciprofloxacin-treated (bottom) \textit{Vibrio} populations. Insets: Viability staining of bacteria expelled from the gut, with green and magenta indicating living and dead cells, respectively. G-H: Dynamics of in vivo \textit{Vibrio} populations untreated (grey lines) and treated with 10 ng/ml ciprofloxacin (blue lines). G: 1D center of mass, normalized to intestine length. H: Total image-derived \textit{Vibrio} abundance. In both (G) and (H), each curve represents a different zebrafish. Vertical dotted lines indicate the time of drug administration to the treatment cohort, $t = 0.67$ hours.}}
%\newpage

From an initial survey of dose-response in rich media, we identified 10 ng/mL ciprofloxacin as an appropriate exposure for \textit{Vibrio} populations. Growth of \textit{Vibrio} in lysogeny broth in the presence of 1 ng/ml ciprofloxacin closely resembles that of the untreated control, while a concentration of 100 ng/ml is largely inhibitory (SI Appendix, Fig. S2A). An intermediate concentration of 10 ng/ml leads to a stable, intermediate optical density. Viability staining (Materials and Methods) after 6 hours of incubation with 10 ng/ml ciprofloxacin identifies 30-80\% of cells as alive (SI Appendix, Fig. S3A and S3B), again consistent with this antibiotic concentration being sufficient to perturb the bacterial population without overwhelming lethality. Growth in the presence of 10 ng/ml ciprofloxacin induces marked changes in cell morphology and motility: treated cells exhibit filamentation, making them considerably longer (mean $\pm$ std. dev. 5.3 $\pm$ 3.1 $\mu$m) than untreated \textit{Vibrio} (2.9 $\pm$ 0.9 $\mu$m) (SI Appendix, Fig. S2B). Swimming speed was also reduced compared to untreated cells (mean $\pm$ std. dev. 11.4 $\pm$ 7.2 $\mu$m/s, untreated  16.9 $\pm$ 11.1 $\mu$m/s) (SI Appendix, Fig. S2C, SI Movies 3 and 4). We note also that 10 ng/ml ciprofloxacin is comparable to levels commonly measured in environmental samples \cite{girardi2011biodegradation}. 



While useful for illuminating the appropriate sub-lethal concentration to further examine, experiments in rich media conditions are not an optimal assay for comparison of in vitro and in vivo antibiotic treatments, as the chemical environments are likely very dissimilar. We therefore assessed effects of ciprofloxacin on bacterial populations in the aqueous environments of the flasks housing the larval zebrafish in comparison to populations in the intestines. In the flask water, as in the intestine, the only nutrients are fish-derived. Oxygen levels are  comparable to those in the larval gut, due to fast diffusion and the animals' small size. Bacteria in flask water therefore constitute a useful baseline against which to compare antibiotic impacts on intestinal populations.

\textit{Vibrio} was associated with germ-free zebrafish at 4 days post-fertilization (dpf) by inoculation of the aqueous environment at a density of 10$^6$ cells/ml (Materials and Methods) and allowed to colonize for 24 hours, which based on previous studies provides ample time for the bacterial population to reach its carrying capacity of approximately $10^5$ cells/gut \cite{wiles_host_2016}. Animals and their resident \textit{Vibrio} populations were then immersed in 10 ng/ml ciprofloxacin for 24 or 48 hours, or left untreated (Fig. 2A and 2B). \textit{Vibrio} abundances in the gut were assayed by gut dissection and plating to measure colony forming units (CFUs) (Materials and Methods). Abundances in the flask water were similarly assayed by plating. We quantified the effect of the antibiotic treatment by computing the ratio of bacterial abundances in the treated and untreated cases, resulting in a normalized abundance (Fig. 2C). After a 24 hour treatment, log$_{10}$-transformed abundances in the flask water dropped by $0.98 \pm 0.4$ (mean $\pm$ std. dev.) compared to untreated controls, or one order of magnitude on average. In contrast, log$_{10}$-transformed intestinal abundances showed a more severe reduction of $1.75 \pm 0.88$ (Fig. 2C), or a factor of approximately 60 on average, suggesting that the intestinal environment amplifies the severity of ciprofloxacin treatment. For the 48 hour treatment, the declines in flask water and intestinal abundances were similarly severe (Fig. 2C). In terms of absolute abundances, pooled data from 24 and 48 hour treatments gives a mean $\pm$ std. dev. of the log$_{10}$-transformed \textit{Vibrio} population of 3.1 $\pm$ 0.9 ($n = 40$), compared to 4.9 $\pm$ 0.5 ($n = 42$) for untreated specimens (Fig. 2D). Unpooled data are similar (SI Appendix, Fig. S3E, S3F).  


To assess the possibility that the intestine makes \textit{Vibrio} more susceptible to ciprofloxacin-induced cell death, we embedded larval zebrafish in a 0.5\% agarose gel, which allowed collection of expelled bacteria. After staining expelled bacterial cells with the viability dyes SYTO9 and propidium iodide, we imaged ejected material. We found  no detectable difference between ciprofloxacin-treated and untreated populations (Fig. 2F, insets). Similarly sizeable fractions of viable and non-viable cells are evident in both ciprofloxacin-treated and untreated populations; however, co-staining of zebrafish host cells hindered exact quantification (SI Appendix, Fig. S4). This result suggests that the ciprofloxacin-induced population decline observed in vivo occurs independent of overt cell death and is a consequence of the response of living bacteria to the intestinal environment. We further note that the dose-response of the intestinal \textit{Vibrio} abundance (SI Appendix, Fig. S5) mirrors the dose-response of the in vitro growth rate, implying that the larval gut does not significantly alter or concentrate ciprofloxacin. This is also consistent with the widespread use of zebrafish larvae as a  pharmacological screening platform, as water soluble chemicals readily enter and leave the animal \cite{rennekamp201515,yoganantharjah2017use}.

To investigate the causes of ciprofloxacin's disproportionately large impact on in vivo bacterial abundance, we used light sheet fluorescence microscopy to directly monitor \textit{Vibrio} populations within the intestine over several hours as they responded to antibiotic exposure. Three-dimensional time-lapse imaging revealed that within hours of ciprofloxacin treatment, large numbers of bacteria became depleted from the anterior-localized planktonic and motile population (SI Movies 5 and 10). Cells were instead found in the mid and distal regions of the gut, where they appeared to be condensed into large multicellular aggregates prior to being expelled from the gut altogether (SI Movies 5 and 11). After 10 hours of exposure, \textit{Vibrio} populations in ciprofloxacin-treated hosts contained large, 3D aggregates localized to the posterior of the intestine, a feature not observed in untreated controls (Fig. 2E and 2F) nor in all previous characterizations of this strain \cite{wiles_host_2016,schlomann_bacterial_2018}. We note also that in vitro, antibiotic-treated \textit{Vibrio} does not form large aggregates (SI Appendix, Fig. S3 and S6, SI Movie 4)

To determine whether the bacterial aggregation observed in vivo stems from a fundamentally different response to antibiotics at the single-cell level or different large-scale consequences of similar cell-level response, we generated in \textit{Vibrio} a genetically encoded fluorescent reporter of the SOS pathway (SI Appendix, Fig. S7, Materials and Methods), a DNA damage repair pathway induced by genotoxic agents such as ciprofloxacin \cite{erill2007aeons,kreuzer2013dna}. Genes in the SOS regulon halt replication and enable DNA repair, and also affect motility and biofilm formation \cite{irazoki2016sos,goneau2015subinhibitory}. In vitro, we found that treatment with 10 ng/ml ciprofloxacin strongly induced \textit{recN}-based SOS reporter activity, with a heterogeneous response across individual cells (SI Appendix, Fig. S3C and S3D). Within the intestine, SOS reporter activity was also heterogeneous, appearing in both planktonic and aggregated cells. Planktonic cells that were SOS-positive appeared more filamented and less motile compared to SOS-negative cells within the same host (SI Movie 6). The activation of the SOS reporter in vitro and in vivo by ciprofloxacin (SI Movie 6 and Fig S3C and S3D) suggests that in both cases a canonical SOS response is involved in the perturbation of \textit{Vibrio} physiology. 

Together, these data begin to reveal a mechanism by which the intestine amplifies the effect of low-dose ciprofloxacin. Individual \textit{Vibrio} cells first undergo an SOS response that is associated with changes in cellular morphology and behavior. In the context of the mechanical activity of the intestine, these molecular and cellular-level changes then give rise to population-level aggregation and spatial reorganization throughout the entire length of the intestine, with the population shifting its center of mass posteriorly (Fig. 2G, $n=4$ per case). This process culminates in the expulsion of large bacterial aggregates from the host, causing a precipitous decline in total bacterial abundance (Fig. 2H).



\subsection{Low-dose ciprofloxacin suppresses small cluster reservoirs associated with intestinal persistence}

In contrast to \textit{Vibrio}, \textit{Enterobacter} is slower growing, non-motile, and naturally forms dense aggregates within the zebrafish intestine. \textit{Enterobacter} populations have an in vivo growth rate of 0.27 $\pm$ 0.05 h$^{-1}$ (mean $\pm$ std. dev, SI Appendix, Fig. S1), compared to 0.8 $\pm$ 0.3 h$^{-1}$ for \textit{Vibrio} \cite{wiles_host_2016}. Based on conventional notions of antibiotic tolerance, we hypothesized that \textit{Enterobacter} would be less affected by ciprofloxacin treatment than the fast growing, planktonic \textit{Vibrio}. However, as detailed below, we found this prediction to be incorrect; \textit{Enterobacter} exhibits an even greater response  to low-dose ciprofloxacin.

\begin{figure}[h]
	\centerline{
		%\includegraphics[width =4.75 in]{abx/fig3.eps}}
		\includegraphics[width =4.75 in]{abx/fig3-eps-converted-to.pdf}}
	\caption{\textbf{Low-dose ciprofloxacin collapses \textit{Enterobacter} populations and suppresses small clusters in vivo.}}{(Continued in footnote.)} 
	\label{fig:fig3}
\end{figure}

{\let\thefootnote\relax\footnote{FIGURE \ref{fig:fig3} (cont'd) A: Normalized abundances (number of colony forming units (CFUs) scaled by untreated medians) of water and gut populations. $N$ values left to right: 4, 20, 4, 20, 4, 19, 4, 20. Water $N$ values denote number of flasks; gut $N$ values denote number of fish. B: Histograms of gut CFUs with pooled data from 24 and 48 h treatments. Counts indicate the number of individual fish with a given log$_{10}$ \textit{Enterobacter} CFUs. Dashed lines indicate the mean of each set, showing a $\sim$1000-fold reduction in intestinal \textit{Enterobacter} abundance in antibiotic-treated fish. C: Total number of bacterial clusters in the intestine, quantified from 3D images (Materials and Methods). D: Maximum intensity projections of 3D images of untreated (top) and ciprofloxacin-treated (bottom) \textit{Enterobacter} populations. Insets: Viability staining of bacteria expelled from the gut, with green and magenta indicating living and dead cells, respectively.}}

We first established in vitro that 25 ng/ml ciprofloxacin produces similar effects on \textit{Enterobacter} growth as did 10 ng/ml exposure on \textit{Vibrio}. With the identical inoculation procedure used for \textit{Vibrio}, log$_{10}$-transformed \textit{Enterobacter} abundance in the flask water dropped by 1.2 $\pm$ 0.4 (mean $\pm$ std. dev.) compared to untreated controls after 24 hours, and dropped by $1.8 \pm 0.2$ after 48 hours (Fig. 3A). These values match well the values for \textit{Vibrio}: 0.98 $\pm$ 0.37 for 24 hours, 1.81 $\pm$ 0.5 for 48 hours. Assays in rich media show a similarly reduced density between the two species (SI Appendix, Fig. S8) and an even lesser degree of cell death and damage in vitro for  \textit{Enterobacter} as compared to \textit{Vibrio}, with a viable fraction of approximately 95\% (SI Appendix, Fig. S9A and S9B). As with \textit{Vibrio}, in vitro growth measurements and viability staining both imply that low-dose ciprofloxacin treatment of \textit{Enterobacter} induces growth arrest rather than widespread lethality.

Strikingly, low-dose ciprofloxacin treatment of fish colonized with \textit{Enterobacter}  (Materials and Methods) resulted in even greater reductions in abundance than in the case of \textit{Vibrio}, with the majority of populations becoming nearly or completely extinct during the assay period (Fig. 3A and 3B). Inoculation, treatment, dissection, and plating were performed as for \textit{Vibrio} (Materials and Methods). Compared to untreated controls, log$_{10}$-transformed intestinal abundances were reduced by 2.3 $\pm$ 1.1 after 24 hours, and by 3.2 $\pm$ 1.0 after 48 hours (Fig. 3A). These reductions in intestinal abundances greatly exceeded the reductions of bacterial abundances in the flask water (Fig 3A). In terms of absolute abundances, pooled data from 24 and 48 hour treatments gives a mean $\pm$ std. dev. of the log$_{10}$-transformed \textit{Enterobacter}  population of 1.5 $\pm$ 1.0 ($n=40$), compared to 4.0 $\pm$ 1.0 ($n=39$) for untreated specimens (Fig. 3B); unpooled data are similar (SI Appendix, Fig. S9C and S9D). 

Live imaging of intestinal populations at single time points revealed approximately 40\% of treated hosts to be devoid or nearly devoid of \textit{Enterobacter}, consistent with the plating-based measurements. In hosts that contained appreciable bacterial populations we observed a clear difference between treated and untreated specimens: \textit{Enterobacter}  populations in ciprofloxacin-treated hosts contained fewer small bacterial clusters and fewer individual planktonic cells than untreated controls (Fig. 3C and 3D). We quantified this distinction using computational image analysis to identify each cluster (Materials and Methods), defining a single cell as a cluster of size one. Bacterial populations in ciprofloxacin-treated animals contained $\sim$80x fewer clusters than untreated animals (Fig. 3C). Viability staining showed that there were no obvious differences in the viable fractions of bacteria expelled from the intestines of untreated and treated hosts (Fig. 3D, insets, SI Appendix, Fig. S10). As with \textit{Vibrio}, these observations suggested that the reduction in \textit{Enterobacter}'s intestinal abundance was independent of cell death.  


Previous studies of other naturally aggregated bacterial species have revealed that large bacterial aggregates are highly susceptible to expulsion from the gut \cite{wiles_host_2016,Logan2018}. To establish whether this is also the case for \textit{Enterobacter} in the absence of low-dose ciprofloxacin treatment, we performed time-lapse 3D imaging (Materials and Methods). Indeed, in 2 out of 5 hosts imaged for 3.5 hours each, we observed events in which the largest bacterial aggregate was abruptly expelled from the intestine (Fig. 4A and SI Movie 7). These time-lapse movies also showed clear examples of cluster aggregation (SI Movie 8), in which single cells and small aggregates appear to come together and fuse, a process that is likely due to the rhythmic intestinal contractions that occur between frames. Importantly, smaller aggregates and planktonic cells that preferentially localize to the intestinal bulb are relatively undisturbed during these expulsion events, save for a few clusters that become incorporated into the large mass during its transit (SI Movie 7). 

Our observations suggest an explanation of how low-dose ciprofloxacin can lead to dramatic drops in \textit{Enterobacter} abundance that moreover illuminates the more general question of how naturally aggregating bacterial species can persist in the vertebrate gut in spite of transport-driven expulsion. We provide both a qualitative and a quantitative description of the relevant dynamics, beginning with the following conceptual model: single cells of \textit{Enterobacter}  replicate to form small clusters, which then aggregate to form larger clusters under the influence of intestinal flow. Large clusters are transported by the rhythmic contractions of the gut \cite{wiles_host_2016,Logan2018,ganz2018} and are stochastically expelled from the host \cite{wiles_host_2016,Logan2018}. The individual bacteria and small clusters that remain within the intestine serve as a reservoir that reseeds the next population, and the process of replication, aggregation, and expulsion repeats. Therefore, persistence  within the intestine requires processes that generate single cells or small clusters, otherwise transport will eventually lead to extinction. This reseeding could take the form of (i) immigration of new cells from the environment, (ii) passive fragmentation of  clusters, or (iii) active fragmentation in which single cells break away from a cluster surface during cell division. Immigration from the environment likely occurs even in established populations, but measurements in larval zebrafish suggest very low rates of immigration \cite{robinson_experimental_2018}. We therefore suspected that more robust mechanisms must promote persistence. Supporting the active fragmentation mechanism, we found in untreated hosts examples of \textit{Enterobacter} populations that contain an abundance of single cells, a single large aggregate, and a lack of mid-sized aggregates (SI Appendix, Fig. S9E). Following low-dose ciprofloxacin treatment, the planktonic cell reservoir associated with resilience to intestinal transport is depleted (Fig. 3C), most likely due to stalled \textit{Enterobacter} division (SI Appendix, Fig. S8), leading to collapse of the resident bacterial population (Fig. 3A and 3B).




 \subsection{A quantitative model of bacterial cluster dynamics}

\begin{figure}%[h!]
\centerline{
	%\includegraphics[width =4.75 in]{abx/fig4.eps}}
	\includegraphics[width =4.75 in]{abx/fig4-eps-converted-to.pdf}}
	\caption{\textbf{Small bacterial clusters are required for recovery after large expulsion events.}}{(Continued in footnote.)}  
	\label{fig:fig4}
\end{figure}

{\let\thefootnote\relax\footnote{FIGURE \ref{fig:fig4} (cont'd) A: Maximum intensity projections of untreated \textit{Enterobacter} populations before (top, $t=2$ hours from the start of imaging) and after (bottom, $t=3$ hours) an expulsion event (See also SI Movie 5). Scale bar = 200 $\mu$m. B: Schematic of a kinetic model of bacterial cluster dynamics, illustrating its four constituent processes. C: Image-derived time-series of \textit{Enterobacter} abundance in five untreated hosts showing sporadic large expulsion events.}}

To solidify and test our conceptual picture, we developed a predictive mathematical model of bacterial cluster dynamics. We describe the framework of the model, its validation, and general insights it provides into perturbations and population stability. Drawing on ideas from non-equilibrium statistical mechanics and soft matter physics, we constructed a general kinetic model that describes the time evolution of a collection of bacterial clusters with varying sizes, illustrated schematically in Fig. 4B. We posit that four processes govern cluster dynamics: aggregation, fragmentation, growth, and expulsion.  Each is described by a kernel that specifies its rate and size dependence: (1) aggregation of a cluster of size $n$ and a cluster of size $m$ occurs with rate $A_{nm}$; (2) fragmentation of a cluster of size $n+m$ into clusters of size $n$ and $m$ occurs with rate $F_{nm}$; (3) growth (due to cell division) of a cluster of size $n$ occurs with rate $G_n$; (4) expulsion (removal by intestinal transport) of a cluster of size $n$ occurs with rate $E_n$. Note that condensation of the population into a single massive cluster poises the system for extinction, for any nonzero $E_n$. The model is non-spatial and is inspired by well established frameworks for nucleation and growth phenomena such as polymer gelation and colloidal aggregation \cite{krapivsky2010kinetic}. For example, sol-gel models describe a transition between dispersed individual units (``sol") and a system-spanning connected network (``gel") in materials capable of polymerization. In the thermodynamic limit of infinite system size, the model  can be studied using standard analytic techniques \cite{krapivsky2010kinetic}. However, unlike polymer solutions and other bulk systems for which the possible number of clusters is effectively unbounded, our intestinal populations are constrained to have at most a few hundred clusters (Fig. 3C), necessitating the use of stochastic simulations (Materials and Methods).

In its general form, the model encompasses a wide range of behaviors that can be encoded in the various functional forms possible for the rate kernels $A_{nm}$ $F_{nm}$, $G_n$, and $E_n$. Based on our observations and theoretical considerations elaborated in the Materials and Methods section, we made the following assumptions: (1) the rate of aggregation between two clusters is independent of their size, $A_{nm} = \alpha$;  (2) fragmentation occurs only by separation of single cells and with a rate that is independent of cluster size, $F_{nm} = \beta$ for $m=1$ and $F_{nm}=0$ otherwise; (3) growth is logistic with a global carrying capacity, $G_n = rn(1-N/K)$ with $N$ the total number of cells, $r$ the per capita growth rate, and $K$, the carrying capacity; (4) expulsion is independent of cluster size, $E_n = \lambda$.  This model contains as special cases various simple models of linear polymers \cite{krapivsky1996transitional} and also resembles recent work modelling chains of \textit{Salmonella typhimurium} cells in the mouse gut \cite{bansept2019enchained}.  As discussed in the SI Appendix, these choices constitute the minimal model consistent with theoretical constraints and experimental phenomenology. More complex models are of course possible, but the requisite increase in the number of adjustable parameters would result in a trivial but meaningless ability to fit the observed data.
  

Even with the assumptions described above, the model needs 5 parameters: rates of aggregation, fragmentation, growth, and dispersal, and a global carrying capacity. However, all of these parameters can be set by experimentally derived values unrelated to cluster size distributions. We measured \textit{Enterobacter}'s per capita growth rate by performing time-lapse imaging of initially germ-free hosts that had been exposed to \textit{Enterobacter} for only 8 hours, capturing the exponential increase of a small founding population (SI Appendix, Fig. S1, SI Movie 9), yielding $r = 0.27 \pm 0.05$ hr$^{-1}$ (mean $\pm$ std. dev across $n=3$ hosts). We identified expulsion events as abrupt collapses in \textit{Enterobacter} abundance from time-lapse images (Fig. 3C, SI Movie 7) and set the expulsion rate equal to the measured collapse rate, $\lambda = 0.11 \pm 0.08$ hr$^{-1}$ (mean $\pm$ standard error, assuming an underlying Poisson process (Materials and Methods)). The model can be simulated to provide the mean and variance of the $\log_{10}$-transformed abundance distribution at a given time for a given set of parameters. Using this approach, we fit static bacterial abundance measurements from dissection and plating at 72 hours post-inoculation (Materials and Methods) to determine the carrying capacity, $K$, and the ratio of fragmentation and aggregation rates, $\beta/\alpha$. As discussed in the Materials and Methods section, the cluster dynamics should depend primarily on the ratio of $\beta/\alpha$ rather than either rate separately. This yielded $\log_{10}K = 5.0 \pm 0.5$ and $\log_{10}\beta/\alpha =  2.5 \pm 0.4$.


The model therefore allows a parameter-free prediction of the size distribution of \textit{Enterobacter} aggregates, plotted in Fig. 5A together with the measured distribution derived from three-dimensional images, averaged across 12 untreated hosts. The two are in remarkable agreement. We also plot, equivalently, the cumulative distribution function $P(\text{size} > n)$, the probability that a cluster will contain greater than $n$ cells, again illustrating the close correspondence between the data and the prediction and validating the model. We emphasize that no information about the cluster size distribution was used to estimate any of the model parameters. We further note that the cluster size distribution is a stringent test of the model's validity. Other cluster models predict different forms, typically with steep tails \cite{krapivsky1996transitional,bansept2019enchained}. The linear chain model of \cite{bansept2019enchained}, for example, leads to an exponential distribution of cluster sizes that does not match the shallower, roughly power-law form of our data.


\begin{figure}%[h!]
\centerline{
	%\includegraphics[width =4.75 in]{abx/fig5.eps}}
	\includegraphics[width =4.75 in]{abx/fig5-eps-converted-to.pdf}}
	\caption{\textbf{A stochastic kinetic model predicts bacterial cluster sizes and generates a phase diagram for in vivo abundance.}}{(Continued in footnote.)}
	\label{fig:fig5}
\end{figure}

{\let\thefootnote\relax\footnote{FIGURE \ref{fig:fig5} A: The distribution of image-derived \textit{Enterobacter} cluster sizes (grey circles) along with the prediction of our stochastic model (purple line). There are no free parameters in the fit; values were fixed by abundance, growth, and expulsion rate measurements independent of cluster size. Parameters: $r = 0.27$ hr$^{-1}$, $\lambda = 0.11$ hr$^{-1}$, $\alpha = 0.1$ hr$^{-1}$, $\beta = 10^{1.5}$ hr$^{-1}$, $K = 10^5$. Error bars on experimental data are standard deviations across hosts. Shaded confidence intervals for the model prediction are bounds from parameter uncertainties. Inset: The same experimental data and model plotted without binning as a reverse cumulative distribution. B: Phase diagram of the log-transformed abundance, $\langle \log_{10}(N+1)\rangle$, showing the extinction transition (white dashed line). From best fit parameter estimates, the in vivo state of untreated \textit{Enterobacter} is overlaid as a grey circle, and 25 ng/ml ciprofloxacin-treated \textit{Enterobacter} as an orange circle; both circles are marked with ``e''. Untreated \textit{Vibrio} is located off the scale, indicated by the arrow, 10 ng/ml ciprofloxacin treated \textit{Vibrio} is overlaid as a cyan circle marked with ``v''. The doses for \textit{Enterobacter} and \textit{Vibrio} were established to be approximately equivalent in vitro. Parameters: $\lambda = 0.11$ hr$^{-1}$, $\alpha =  0.1$ hr$^{-1}$, $K = 10^5$ were fixed; $r$ and $\beta$ were varied on logarithmic grids.}}

 \subsection{The abundance phase diagram and extinction transition}

Our kinetic model provides insights into the consequences of low-dose antibiotic perturbations on gut bacterial populations. We consider a general phase diagram of possible growth, fragmentation, aggregation, and expulsion rates, and then situate \textit{Enterobacter} in this space. For simplicity of illustration, we consider a two-dimensional representation with one axis being the ratio of the fragmentation and aggregation rates, $\beta/\alpha$, and the other being the ratio of the growth and expulsion rates, $r/\lambda$ (Fig. 5B). As noted above and in the Materials and Methods section, the model in the regime studied should depend on the ratio $\beta/\alpha$ rather than on $\beta$ or $\alpha$ independently. However, the roles of $r$ and $\lambda$ are not simply captured by their ratio. The expulsion rate nonetheless provides a scale to which to compare the growth rate, $r$, and we plot Fig. 5B using  $r/\lambda$  calculated for fixed $\lambda = 0.11$ hr$^{-1}$, the measured value. For completeness, we show a three-dimensional $r,  \lambda, \beta/\alpha$ phase diagram as SI Appendix, Figure S11E and S11F. We numerically calculated the steady state phase diagram of the model (Materials and Methods) and show in Figure 5B the mean log-transformed abundance, $\langle\log_{10}(N+1)\rangle$. The regime of extinction $(N=0)$ is evident (dark purple, with dashed white boundary ). 


The data-derived parameter values place untreated intestinal \textit{Enterobacter} fairly close to the extinction transition (Fig. 5B). An antibiotic-induced growth rate reduction of approximately 5x is sufficient to cross the boundary to the $N=0$ regime (i.e. to extinction), moving downward in Fig. 5B. This growth rate reduction, or an equivalent increase in death rate, reflects the conventional view of antibiotic effects.  An approximately 300x reduction in the balance between fragmentation and aggregation spurs an alternative path to extinction, moving leftward in Fig. 5B, reflecting a distinct mechanism resulting solely from changes in physical structure. The extinction transition in this direction corresponds to the condensation of the population into a single cluster, reminiscent of gelation phase transitions in polymer systems. As described above, low-dose ciprofloxacin causes a strong reduction in the number of small bacterial clusters, lowering $\beta$ and possibly also $r$ if fragmentation and individual cell division are linked. Conservatively assuming an equal effect along both axes, and fitting simulations to the 24 hour treatment abundances (Materials and Methods), we find that the antibiotic reduces $r$ and $\beta/\alpha$ by $\sim$10x. This drives the bacterial system through the phase boundary and well into the extinction regime (Fig. 5B, orange circle), consistent with our observations. 

In contrast to \textit{Enterobacter}, treatment of \textit{Vibrio} with ciprofloxacin does not lead to widespread extinction after 48 hours, suggesting that treated populations either lie safely at a new steady state away from the extinction boundary, or are close enough to the transition so that dynamics are slow. To estimate model parameters for ciprofloxacin-treated \textit{Vibrio}, we performed a two parameter fit of $(\beta/\alpha, r)$ to the 24 hour treatment abundances. Because of \textit{Vibrio}'s large population size ($\sim 10^5$ clusters), we modified the stochastic simulation procedure using a tau-leaping algorithm (Materials and Methods, SI Appendix, Fig. S12). We indeed find ciprofloxacin-treated \textit{Vibrio} is located close to but safely inside the extinction boundary (Fig. 5B). Untreated \textit{Vibrio} populations show no appreciable multicellular aggregation and are located off-scale far to the upper-right side of the phase diagram (Fig. 5B, arrow).  



 \section{Discussion}

We have discovered that sublethal levels of a commonly used antibiotic can reduce the intestinal abundance of bacterial populations much more severely than would be predicted from in vitro responses, and that this amplification is a consequence of drug-induced changes to the bacterial groups' spatial architecture. Contrary to conventional notions of antibiotic tolerance, largely derived from in vitro studies, reductions in bacterial abundances were greater for the slow-growing, aggregated \textit{Enterobacter} species than for the fast-growing, planktonic \textit{Vibrio}. Live imaging revealed drug-induced increases in bacterial cohesion that, coupled to gut mechanical activity, lead to the expulsion of viable bacterial cells. The microscopic details of this cohesion, likely involving cell wall characteristics, mechanical compression by the gut wall and fluid flows, and perhaps intestinal mucus rheology, remain to be explored.

Notably, the underlying processes of bacterial aggregation and host intestinal transport are found throughout the animal kingdom, suggesting a general relevance beyond zebrafish that may explain, for example, data on weak antibiotics having strong effects on mammalian microbiomes \cite{cho2012antibiotics,schulfer2019impact}. Of course, chemical perturbations in more anatomically complex animals or non-gnotobiotic animals that house hundreds of resident bacterial species will undoubtedly involve additional processes beyond those uncovered here. We note, however, that responses to intestinal flow will influence bacterial population dynamics regardless of ecological complexity, and that our choice of model bacterial species spans the extremes of highly planktonic and highly cohesive strains, further implying generality. In the larval zebrafish, enhanced bacterial susceptibility to transport leads to expulsion from the gut. In larger or more complex intestines this may take the form of displacement from one region to a more distal region, with a corresponding shift in local nutrients or competitors, in addition to expulsion from the gut altogether.

The concentrations of ciprofloxacin examined here are commonly found in environmental samples, indicating a potentially widespread perturbation of animal gut microbiota due to antibiotic contaminants. In addition, the expulsion of live, antibiotic-exposed bacteria from animal intestines through the aggregation-based processes described here suggests a potential mechanism for enhanced spread of antibiotic resistance. This possibility is bolstered by our observation that in addition to aggregation, ciprofloxacin-treated cells undergo an active SOS response, which has been shown to promote mutation and horizontal gene transfer \cite{baharoglu2011vibrio,guerin2009sos,beaber2004sos}. Together, these observations underscore recent concerns about the public health risk posed by antibiotic contaminants in the environment \cite{national2018environmental}.

Our biophysical model of aggregation, fragmentation, growth, and expulsion describes our data well and provides testable predictions. It is remarkable, given the chemical and morphological complexity of even the larval zebrafish gut, that such a minimal model can accurately predict emergent properties such as the size distribution of bacterial aggregates. That this works is an indication of the power of theories of soft condensed matter physics, whose generality may prove useful in understanding the gut microbiome. Furthermore, our model supplies a framework for a quantitative understanding of gut microbial homeostasis in general. Like recent work modelling antibody-mediated enchaining of \textit{Salmonella} cells in the mouse gut \cite{bansept2019enchained}, our model implies that the physical processes of bacterial cluster formation and fragmentation play central roles in large-scale microbiota stability. We suggest that our cluster-dynamics model, validated by quantitative agreement between  predictions and in vivo data (Fig. 5A), may prove useful in less tractable host species such as mice and humans. Without live imaging or non-invasive sampling, it is challenging to estimate kinetic properties of microbial populations, such as aggregation rates. However, advances in histological sample preparation \cite{tropini_gut_2017} can preserve bacterial aggregates and yield cluster size distributions; inverting our model, such distributions can reveal the underlying in vivo bacterial kinetics.

Regarding antibiotics, the main prediction of our model is that naturally aggregated, slow growing bacteria will be impacted more severely than fast growing, planktonic species by equivalent low-dose antibiotic perturbations. This is contrary to conventional wisdom that links bacterial tolerance to reduced growth and increased aggregation  \cite{walters2003contributions,fux2005survival}, which stems from studies of antibiotic exposure in static or well-mixed environments. We find that in the intestine, where bacteria can be removed through fluid flow, there exist critical values of aggregation, fragmentation, growth, and expulsion rates, beyond which sustainable colonization becomes impossible (Fig. 5B). Naturally aggregated and slow-growing species are situated closer to this extinction phase boundary and are therefore more easily driven to population collapse by low-dose antibiotic perturbations. Intriguingly, new meta-omics methods \cite{korem2015growth} can be used to estimate in vivo growth rates of mammalian gut microbes, which would be interesting to correlate with antibiotic responses. We note in addition that inter-bacterial competition in the gut can be influenced by clustering and susceptibility to intestinal transport \cite{wiles_host_2016, Logan2018}, suggesting that competition outcomes could be altered by antibiotic treatment if changes in aggregation properties are different for different species. A final prediction of our model is that intestinal transport, which has been linked to microbiota composition \cite{cremer2017effect}, will influence the effects of low-dose antibiotic perturbations on microbial community composition. Combining pharmacological manipulations of intestinal transport with antibiotic treatments may therefore lead to novel strategies for precision engineering of the gut microbiome. 


 \section{Materials and Methods}
 \subsection{Animal care}
All experiments with zebrafish were done in accordance with protocols approved by the University of Oregon Institutional Animal Care and Use Committee and following standard protocols \cite{westerfield_zebrafish_2007}. 

 \subsection{Gnotobiology}
Wild-type (AB$\times$TU strain) zebrafish were derived germfree (GF) and colonized with bacterial strains as previously described \cite{melancon_best_2017} with slight modifications elaborated in the SI Appendix. 

 \subsection{Bacterial strains and culture}
\textit{Vibrio cholerae} ZWU0020 and \textit{Enterobacter cloacae} ZOR0014 were originally isolated from the zebrafish intestine \cite{stephens_composition_2016}. Fluorescently marked derivatives of each strain were previously generated by Tn\textit{7}-mediated insertion of a single constitutively expressed gene encoding dTomato \cite{wiles_modernized_2018}. We note that all plating- and imaging-based experiments performed in this study were done using fluorescently marked strains, which carry a gentamicin resistance cassette, with the exception of experiments in which fluorescent dyes were used to assess viability of cells. Archived stocks of bacteria were maintained in 25\% glycerol at -80$^{\circ}$C. Prior to experiments, bacteria were directly inoculated from frozen stocks into 5 ml LB media (10 g/L NaCl, 5 g/L yeast extract, 12 g/L tryptone, 1 g/L glucose) and grown for $\sim$16 hours (overnight) shaking at 30$^{\circ}$C.
%

 \subsection{Culture-based quantification of bacterial populations} 
Dissection of larval guts was done as described previously \cite{milligan-myhre_study_2011}, with slight modifications elaborated in the SI Appendix. To compare the effect of ciprofloxacin on populations in the intestine and in the flask water, we normalized treated abundances by the corresponding untreated median abundance (Fig. 2C and 3A). To account for variation in untreated bacterial dynamics between weekly batches of fish, we performed the normalization within each batch. Unnormalized data is available in the SI Dataset.
%
%
 \subsection{Light sheet fluorescence microscopy of live larval zebrafish}
Live imaging of larval zebrafish was performed using a custom-built light sheet fluorescence microscope previously described in detail \cite{jemielita_spatial_2014}, with slight modifications elaborated in the SI Appendix.

%
 \subsubsection{Viability staining of expelled aggregates:}  Germ-free larval zebrafish were colonized with wild type \textit{Vibrio} or \textit{Enterobacter} (without fluorescent markers) for 24 hours and then mounted into agarose plugs using small glass capillaries identically to the imaging procedure (above). Individual capillaries were suspended into isolated wells of a 24-well tissue culture plate filled with embryo media containing anesthetic or anesthetic + ciprofloxacin (10 ng/ml for \textit{Vibrio}, 25 ng/ml for \textit{Enterobacter}) and the larvae were extruded from the capillaries. Fish remained mounted for 24 hours, during which expelled bacteria remained caught in the agarose plug. After treatment, fish were pulled back into the capillaries and transferred to smaller wells of a 96 well plate containing embryo media, anesthetic, and the LIVE/DEAD BacLight Bacterial Viability stains SYTO9 and propridium iodide. Fish were stained according to kit instructions, with the exception of the incubation period being extended from 15 to 30 min to account for potential issues with the aggregate nature of the cells \cite{netuschil2014confusion}.  Following staining, fish were pulled again into the capillaries and transferred to the light sheet microscope for imaging. As shown in the SI Appendix, Figures S4 and S10, zebrafish cells stain in addition to bacterial cells, precluding accurate quantification of viable fractions.

 \subsection{Image analysis}

Bacteria were identified in three-dimensional light sheet fluorescence microscopy images using a custom MATLAB analysis pipeline previously described \cite{jemielita_spatial_2014,schlomann_bacterial_2018}, with minor changes elaborated in the SI Appendix.


 \subsection{Kinetic model and stochastic simulations}
Simulation details are provided in the SI Appendix. In brief, Gillespie's direct method \cite{gillespie1977exact} was used to simulate stochastic aggregation, fragmentation, and expulsion events, while growth was treated as deterministic. To simulate \textit{Vibrio} populations, direct stochastic simulation becomes intractable due to the large number of clusters ($\sim 10^5$ single cells). We therefore implemented a modified tau-leaping algorithm \cite{gillespie2001approximate} that facilitates large simulations. We opted for a straightforward fixed $\tau$ method and empirically determined an optimal value of $\tau = 0.001$ h (SI Appendix, Fig. S12A,B). All simulations were written in MATLAB and code is available at \texttt{https://github.com/bschloma/gac}.






 \section{Acknowledgements}
We thank Rose Sockol and the University of Oregon Zebrafish Facility staff for fish husbandry. Research was supported by an award from the Kavli Microbiome Ideas Challenge, a project led by the American Society for Microbiology in partnership with the American Chemical Society and the American Physical Society and supported by The Kavli Foundation. Work was also supported by the National Science Foundation under Awards 1427957 (R.P.) and 0922951 (R.P.), the M.J. Murdock Charitable Trust, and the National Institutes of Health (http://www.nih.gov/), under Awards P50GM09891 and P01GM125576-01 to K.G. and R.P., F32AI112094 to T.J.W., and T32GM007759 to B.H.S. The content is solely the responsibility of the authors and does not represent the official views of the NSF, National Institutes of Health, or other funding agencies. This work benefited from access to the University of Oregon high performance computer, Talapas.


\section{Supporting Materials and Methods}

\subsection*{Animal care}
All experiments with zebrafish were done in accordance with protocols approved by the University of Oregon Institutional Animal Care and Use Committee and following standard protocols \cite{westerfield_zebrafish_2007}. 

\subsection*{Gnotobiology}
Wild-type (AB$\times$TU strain) zebrafish were derived germfree (GF) and colonized with bacterial strains as previously described \cite{melancon_best_2017} with slight modifications. Briefly, fertilized eggs from adult mating pairs were harvested and incubated in sterile embryo media (EM) containing ampicillin (100 $\mu$g/ml), gentamicin (10 $\mu$g/ml), amphotericin B (250 ng/ml), tetracycline (1 $\mu$g/ml), and chloramphenicol (1 $\mu$g/ml) for 6 hours. Embryos were then washed in EM containing 0.1\% polyvinylpyrrolidone-iodine followed by EM containing 0.003\% sodium hypochlorite. Sterilized embryos were distributed into T25 tissue culture flasks containing 15 ml sterile EM at a density of one embryo per milliliter and incubated at 28 to 30$^{\circ}$C prior to bacterial colonization. Embryos were sustained on yolk-derived nutrients and were not fed during experiments. For bacterial mono-association, bacteria were first grown overnight in lysogeny broth (LB) with shaking at 30$^{\circ}$C and were prepared for inoculation by pelleting 1 ml of culture for 2 min at 7,000$\times g$ and washing once in sterile EM. Bacterial strains were individually added to the water column of single flasks containing 4-day-old larval zebrafish at a final density of 10$^6$ bacteria/ml. For antibiotic treatment, drugs were added at the indicated working concentration directly to flask containing animals that had been colonized 24 hours prior. 

\subsection*{Bacterial strains and culture}
\textit{Vibrio cholerae} ZWU0020 and \textit{Enterobacter cloacae} ZOR0014 were originally isolated from the zebrafish intestine \cite{stephens_composition_2016}. Fluorescently marked derivatives of each strain were previously generated by Tn\textit{7}-mediated insertion of a single constitutively expressed gene encoding dTomato \cite{wiles_modernized_2018}. We note that all plating- and imaging-based experiments performed in this study were done using fluorescently marked strains, which carry a gentamicin resistance cassette, with the exception of experiments in which fluorescent dyes were used to assess viability of cells. Archived stocks of bacteria were maintained in 25\% glycerol at -80$^{\circ}$C. Prior to experiments, bacteria were directly inoculated from frozen stocks into 5 ml LB media (10 g/L NaCl, 5 g/L yeast extract, 12 g/L tryptone, 1 g/L glucose) and grown for $\sim$16 hours (overnight) shaking at 30$^{\circ}$C.

\subsection*{Generation of a fluorescent SOS reporter}
To identify a suitable promoter within the \textit{Vibrio} ZWU0020 genome (\texttt{https://img.jgi.doe.gov/m/}, IMG genome ID: 2522572152) for creation of a genetically encoded fluorescent DNA-damage `SOS' reporter, we scanned the upstream regions of each gene for consensus gammaproteobacterial `SOS boxes' (\texttt{CTGTN$_8$ACAG}) that serve as binding sites for the repressor LexA (Fig. S7A and S7B) \cite{erill2007aeons}. Of the genes identified, the promoter of the gene \textit{recN} (IMG gene ID: 2705597027) was an ideal candidate for three main reasons: 1) it contains multiple SOS boxes (2 consensus and 2 with 2 mismatches), which is an arrangement that is potentially associated with tight/graded regulation \cite{kreuzer2013dna}; 2) the \textit{recN} promoter is highly conserved among closely related \textit{V. cholerae} strains as well as other non-\textit{Vibrio} gammaproteobacterial lineages, suggesting that \textit{recN} is a bona fide representative of the SOS response; and 3) \textit{recN} is one of the most highly expressed genes in response to DNA damaging agents in both \textit{E. coli} and \textit{V. cholerae} \cite{courcelle2001comparative,Krin2018}, likely due to its multiple near-consensus -10 promoter sequences.

We rationally designed a \textit{recN}-based fluorescent SOS reporter by fusing the 100bp \textit{recN} promoter region to an open reading frame (ORF) encoding superfolder green fluorescent protein (sfGFP) (Fig. S7C). In addition, we incorporated an epsilon enhancer and consensus Shine-Dalgarno sequence within the 5' untranslated region (UTR) to help ensure robust translation of the reporter gene \cite{wiles_modernized_2018,olins1989novel,olins1988t7}, and incorporated the synthetic transcriptional terminator L3S2P21 into the 3' UTR \cite{chen2013characterization}. We built the construct using polymerase chain reaction (PCR) and synthetic oligonucleotides. Primer WP97 (containing the \textit{recN} promoter and 5' UTR; 5'-\texttt{\seqsplit{TGAATGCATTAAAAGTGACCAAAAAATTTTACCTGAGTGACTTTACTGTATAAAGAAACAGTATAAACTGTTTAAACATACAGTATTGGTTAATCATACAGGTGCAAACTTAACTTTATCAAGGAGACTAAATCATGAGCAAGGGCGAGGAGCT}}-3') and primer WP98 (containing the 3' UTR; 5'-\texttt{\seqsplit{TGAACTAGTAAAACGAAAAAAGGCCCCCCTTTCGGGAGGCCTCTTTTCTGGAATTTTTATCACTTGTACAGCTCGTCCATG}}-3') were used to PCR-amplify sfGFP from the source plasmid pXS-sfGFP \cite{wiles_modernized_2018}. Engineered restriction sites flanking the amplicon (NsiI and SpeI) were then used to insert the construct into a variant of the Tn\textit{7} delivery vector pTn\textit{7}xKS, which also harbors a constitutively expressed \textit{dTomato} gene for tracking all bacterial cells (Fig. S7D) \cite{wiles_modernized_2018}. The resulting dual-reporter construct was then inserted into the ZWU0020 genome as previously described \cite{wiles_modernized_2018}. To verify reporter activity, disk diffusion assays were performed on agar plates with the genotoxic agent mitomycin C and, as a control, the cell wall-targeting beta-lactam antibiotic ampicillin (Fig. S7E). Mitomycin C induced robust expression of sfGFP whereas ampicillin did not.  




\subsection*{In vitro characterization of antibiotics}

\textit{Growth kinetics:} Growth kinetics of bacterial strains in vitro were measured using a FLUOstar Omega microplate reader. Prior to growth measurements, bacteria were grown overnight in 5 ml LB media at 30$^{\circ}$C with shaking. The next day, cultures were diluted 1:100 into fresh LB media with or without the indicated antibiotic and dispensed in quadruplicate (200 $\mu$l/well) into a sterile 96-well clear flat-bottom tissue culture-treated microplate. Absorbance at 600 nm was then recorded every 30 min for $\sim$16 hours at 30$^{\circ}$C with shaking. Growth rates were estimated by fitting a logistic growth curve to OD values, starting at manually defined points marking the end of lag phase.

\textit{Viability:} Cultures of \textit{Vibrio} ZWU0020 or \textit{Enterobacter} ZOR0014 were grown overnight in LB at 30$^{\circ}$C with shaking. The next day, 1:100 dilutions were made in fresh LB media containing either ciprofloxacin (\textit{Vibrio}: 10 ng/ml, \textit{Enterobacter}: 25 ng/ml) or no drug. Cultures were incubated at 30$^{\circ}$C with shaking for 6 hours prior to being stained using a LIVE/DEAD BacLight Bacterial Viability Kit according to manufacturer's instructions. Culture/stain mixtures were diluted 1:10 in 0.7\% saline and imaged using a Leica MZ10 F fluorescence stereomicroscope equipped with a 2.0X objective and a Leica DFC365 FX camera. Images were captured using standard Leica Application Suite software. Bacteria were identified in images with intensity-based region finding following difference of gaussians filtering. Cells stained in both SYTO9 and propidium iodide were identified as overlapping regions in the two color channels. Analysis code was written in MATLAB.

\textit{Cell length and swimming speed}: Dense overnight cultures of \textit{Vibrio} ZWU0020 were diluted 1:100 in fresh LB media alone or with 10 ng/ml ciprofloxacin and incubated at 30$^{\circ}$C with shaking for 4 h. Bacteria were then imaged on a Nikon TE2000 inverted fluorescence microscope between a slide and a coverslip using a 60X oil immersion objective and a Hamamatsu ORCA CCD camera (Hamamatsu City, Japan). Movies were taken within 60 seconds of mounting at an exposure time of 30 ms, resulting in a frame rate of 15 frames/sec, and had a duration of approximately 7 seconds. Bacteria in the resulting movies were identified with intensity-based region finding and tracked using nearest-neighbor linking. Analysis code was written in MATLAB. Five movies were taken per treatment case. For untreated length analysis, $n = 2291$ bacteria were quantified; for ciprofloxacin-treated length analysis, $n = 963$. For untreated speed analysis, $n = 833$ bacteria; for ciprofloxacin-treated speed analysis, $n = 531$.

\textit{Vibrio SOS reporter activity:} \textit{Vibrio} ZWU0020 carrying the fluorescent SOS reporter was grown overnight in LB at 30$^{\circ}$C with shaking. The next day, 1:100 dilutions were made in fresh LB media containing either 10 ng/ml ciprofloxacin, 400 ng/ml mitomycin C, 10 $\mu$g/ml ampicillin, or no drug. Cultures were then grown overnight ($\sim$16 h) at 30$^{\circ}$C with shaking. The next day, cultures were diluted 1:43 in 80\% glycerol (as an immobilizing agent) and imaged with a Nikon Eclipse Ti inverted microscope equipped with an Andor iXon3 888 camera using a 40x objective and 1.5x zoom. Bacteria were identified in images with gradient-based region finding, using a Sobel filter, following difference of gaussians filtering. Analysis code was written in MATLAB. As expected, the two DNA targeting drugs, ciprofloxcain and mitomycin C, induced the SOS response in subpopulations of cells, while the cell-wall targeting drug ampicillin did not. In computing SOS-positive fractions, filamented cells were counted as single cells.

\subsection*{Culture-based quantification of bacterial populations} 
Dissection of larval guts was done as described previously \cite{milligan-myhre_study_2011}. Dissected guts were harvested and placed in a 1.6 ml tube containing 500 $\mu$l sterile 0.7\% saline and $\sim$100 $\mu$l 0.5 mm zirconium oxide beads. Guts were then homogenized using a bullet blender tissue homogenizer for $\sim$25 seconds on power 4. Lysates were serially plated on tryptic soy agar (TSA) and incubated overnight at 30$^{\circ}$C prior to enumeration of CFU and determination of bacterial load. Typically an overnight incubation is sufficient to recover all viable cells; however, we note that ciprofloxacin treatment results in delayed colony growth on agar plates (likely due to growth arrest induced by DNA-damage). We empirically determined that, in the case of ciprofloxacin treatment, an incubation period 72 hours was required for complete resuscitation of viable cells on agar plates. For all culture-based quantification of bacterial populations in this study, the estimated limit of detection is 5 bacteria/gut and the limit of quantification is 100 bacteria/gut. Plating data plotted are pooled from a minimum of two independent experiments. Samples with zero countable colonies on the lowest dilution were set to the limit of detection prior to plotting and statistical analysis. Enumeration of flask water abundances by plating was performed identically to gut abundances, including the 72 hour incubation period.

\subsubsection*{Comparing antibiotic treatments between intestinal populations and flask water populations:}  To compare the effect of ciprofloxacin on populations in the intestine and in the flask water, we normalized treated abundances by the corresponding untreated median abundance (Fig. 2C and 3A). To account for variation in untreated bacterial dynamics between weekly batches of fish, we performed the normalization within each batch. Unnormalized data is available in the Supplemental Data File.


\subsection*{Light sheet fluorescence microscopy of live larval zebrafish}
\subsubsection*{Imaging intestinal bacteria:}  Live imaging of larval zebrafish was performed using a custom-built light sheet fluorescence microscope previously described in detail \cite{jemielita_spatial_2014}. Larvae are anesthetized with MS-222 (Tricane) and mounted into small glass capillaries containing 0.5\% agarose gel by means of a metal plunger. Larvae are then suspended vertically in an imaging chamber filled with embryo media and anesthetic and extruded out of the capillary such that the set agar plug sits in front of the imaging objective. The full intestine volume ($\sim$1200 $\times$ 300 $\times$ 150 microns) is imaged in four subregions that are registered in software after imaging. The imaging of a full intestine volume sampled at 1-micron steps between $z$-planes is imaged in $\sim$45 seconds. Excitation lasers at 488 and 561 nm wavelengths were tuned to a power of 5 mW prior to entering the imaging chamber. A 30 ms exposure time was used for all 3D scans and 2D movies. Time lapse imaging was performed overnight, except for the 3.5 hour imaging of \textit{Enterobacter} (Fig. 3C), which occurred during the day.

\subsubsection*{Viability staining of expelled aggregates:}  Germ-free larval zebrafish were colonized with wild type \textit{Vibrio} or \textit{Enterobacter} (without fluorescent markers) for 24 hours and then mounted into agarose plugs using small glass capillaries identically to the imaging procedure (above). Individual capillaries were suspended into isolated wells of a 24-well tissue culture plate filled with embryo media containing anesthetic or anesthetic + ciprofloxacin (10 ng/ml for \textit{Vibrio}, 25 ng/ml for \textit{Enterobacter}) and the larvae were extruded from the capillaries. Fish remained mounted for 24 hours, during which expelled bacteria remained caught in the agarose plug. After treatment, fish were pulled back into the capillaries and transferred to smaller wells of a 96 well plate containing embryo media, anesthetic, and the LIVE/DEAD BacLight Bacterial Viability stains SYTO9 and propridium iodide. Fish were stained according to kit instructions, with the exception of the incubation period being extended from 15 to 30 min to account for potential issues with the aggregate nature of the cells \cite{netuschil2014confusion}.  Following staining, fish were pulled again into the capillaries and transferred to the light sheet microscope for imaging. As shown in Figures S4 and S10, zebrafish cells stain in addition to bacterial cells, precluding accurate quantification of viable fractions.

\subsection*{Image analysis}

Bacteria were identified in three-dimensional light sheet fluorescence microscopy images using a custom MATLAB analysis pipeline previously described \cite{jemielita_spatial_2014,schlomann_bacterial_2018}, with minor changes. In brief, small objects (single cells and small aggregates) are identified using difference of Gaussians filtering. False positives are rejected with a combination of intensity thresholding (mostly noise) and manual removal (mostly host cells). Large aggregates are identified with a graph cut algorithm \cite{boykov_experimental_2004} that is seeded with either an intensity-based mask or a gradient-based mask. The average intensity of a single cell is estimated as the mean intensity of small objects, which is then used to estimate the number of cells contained in larger clusters by normalizing the total fluorescence intensity of each cluster. Spatial distributions along the length of the gut are computed using a manually drawn line drawn that defines the gut's center axis.


\subsection*{Kinetic model and stochastic simulations}
\subsubsection*{Choosing rate kernels:  }Our approach to choosing the size dependence of the rate parameters was to pick the simplest kernels consistent with key experimental observations. The first key observation, made in past work \cite{jemielita_spatial_2014,wiles_host_2016}, was that in between the expulsion of large aggregates population growth is well-described by a deterministic logistic function. Therefore, we chose a logistic growth kernel. The second key observation was that we occasionally encountered populations consisting of just a single, large aggregate and many single cells (Fig. S9E), which suggests that active fragmentation of single cells, most likely during growth phases, is the dominant fragmentation process. This notion is supported by time-lapse images of initial growth (Supplemental Movie 9) that depicts the creation of single cells during growth, in addition to the growth of three dimensional aggregates. Based on these observations, we made the assumption that single cell fragmentation is the sole fragmentation process, leading to what is known in other contexts as a ``chipping'' kernel \cite{krapivsky2010kinetic}. Beyond the chipping assumption, we had little evidence that informed how single cell fragmentation depends on the size of the aggregate, so we opted for the simplest choice of a constant, size-independent rate. Similarly for aggregation and expulsion, the size dependence of the rates is likely determined by complicated and uncharacterized fluid mechanical interactions of bacterial clusters in peristaltic-like flow, which we parsimoniously replace with a simple constant kernel for both processes. In aggregated populations, since it is only the loss of the largest clusters (of size $\mathcal{O}(K)$) that significantly impacts the system, we expect that it is the expulsion rate for these largest clusters that matters, rather than how the expulsion rate scales with cluster size. To test this notion, we ran simulations in which the expulsion rate scaled as a power of the cluster size, $n$, according to $\lambda(n) = \lambda(n/K)^{\nu}$, and varied the exponent $\nu$. This ansatz keeps the expulsion rate of clusters of size $K$ fixed for all values of $\nu$.  The result is that the cluster size distribution does not change within uncertainty values (Fig. S13), indicating that this approximation is valid.

For reference, we note that with these choices the model can be summarized by the following Smoluchowski equation, which describes the time evolution of the concentration of clusters of size $n$, $c_n$, in the thermodynamic limit of infinite system size:

\begin{align}
\dot{c}_n = \text{ }& \frac{\alpha}{2}\sum_{m=1}^{n} c_{n-m}c_m - \alpha c_n \sum_{m=1}^{\infty}c_m+\beta(c_{n+1}- c_n) + \beta\delta_{n,1}\sum_{m=1}^{\infty}c_m\nonumber\\[6pt]
&+r\left(1-\frac{\sum_{m=1}^{\infty}mc_m}{K}\right)\left[(n-1)c_{n-1} - nc_n\right]  - \lambda c_n. 
\end{align}

\noindent The four rate parameters are $\alpha$ (aggregation), $\beta$ (fragmentation), $r$ (growth), and $\lambda$ (expulsion), and $K$ is the carrying capacity. In the last term of the first line,  $\delta_{n,1}$ is the Kronecker delta with second argument equal to $1$. Of note, the first line of equation (1), containing just aggregation and fragmentation terms, was previously studied as a model of polymer chains and was shown to exhibit interesting non-equilibrium steady states and scaling behaviors that are due to the breaking of detailed balance by the chipping kernel \cite{krapivsky1996transitional}. In our system detailed balance is also broken, but for a different reason: our ``monomers''---single cells---are alive and self-replicating.

\textit{Simulations}: As each zebrafish intestine contains at most a few hundred bacterial clusters, finite size effects and stochasticity impact cluster statistics, so we implemented the model as a hybrid deterministic-stochastic simulation that follows the time evolution of individual clusters. Gillespie's direct method \cite{gillespie1977exact} was used to simulate stochastic aggregation, fragmentation, and expulsion events. Growth was treated as deterministic. Once the time until next stochastic reaction, $\tau$, was determined according to the Gillespie algorithm, integration was performed with the Euler method from time $t$ to $t+\tau$ using a time step $\Delta t = \text{min}(\tau, 0.1 \text{ hr})$. 

To simulate \textit{Vibrio} populations, direct stochastic simulation becomes intractable due to the large number of clusters ($\sim 10^5$ single cells). We therefore implemented a modified tau-leaping algorithm \cite{gillespie2001approximate} that facilitates large simulations. We opted for a straightforward fixed $\tau$ method and empirically determined that a value of $\tau = 0.001$ h produced no observable differences in cluster size and abundance distributions compared to direct stochastic simulation (SI Appendix, Fig. S12A,B).

All simulations were written in MATLAB and code is available at \texttt{https://github.com/bschloma/gac}.

\subsection*{Parameter inference}
The kinetic model presented in the main text has 5 parameters: rates of growth, expulsion, aggregation, and fragmentation, along with an overall carrying capacity. As discussed in the main text, we directly measured \textit{Enterobacter}'s growth rate and expulsion rate through time-lapse imaging. The uncertainty of the expulsion rate was estimated by the standard error, using the previously validated assumption that the expulsion of large aggregates follows a Poisson process \cite{wiles_host_2016}:

\be
\text{SE}_{\lambda} = \frac{\sqrt{\text{mean number of expulsions}}}{(\text{imaging time})\times\sqrt{\text{number of fish}}}.
\ee

For the remaining parameters, we developed a method to infer them from the distribution of abundances obtained from dissection and plating assays. In a regime where aggregation and fragmentation are fast compared to expulsion, we expect the system to locally reach a quasi-steady state in between expulsions of the largest aggregates. As such, we expect cluster statistics to depend primarily on the ratio of fragmentation to aggregation, $\beta/\alpha$, rather than on each rate independently. This confirmed in simulations (Fig. S11A and S11B). Therefore, the number of parameters to be estimated is reduced to two: $\beta/\alpha$ and $K$. 

\textit{Untreated Enterobacter}:  We fixed $\alpha = 0.1$ hr$^{-1}$ and performed a grid search in $\beta$ and $K$ on a logarithmic grid, simulating the model multiple trials for each pair of $(\beta,K)$. The number of trials decreased with increasing $\beta$, from 1000 to 10. Each simulation started from 10 single cells and ran for a simulated time of 64 hours, modeling our 72 hour colonization data with an 8 hour colonization window. To model static host-host variation, we drew each carrying capacity from a log-normal distribution with a standard deviation of 0.5 decades. This is the standard deviation of the untreated \textit{Vibrio} abundance distribution (Fig. S3F), which is an appropriate measure of static host-host variation because untreated \textit{Vibrio} does not form large aggregates and therefore does not experience large, stochastic population collapses due to aggregate expulsion. We then compared the mean ($\mu$) and variance ($\sigma$) of the simulated, log-transformed abundances $\log_{10}(N+1)$ with the values for our plating data ($\hat{\mu}$ and $\hat{\sigma}$, respectively), quantifying error using 

\be
\chi^2 = (\mu - \hat{\mu})^2 + (\sigma - \hat{\sigma})^2.
\ee

A heat map of $\chi^2$ shows well-defined edges for the minimum values of the fit parameters (Fig. S11C). However, the inference is poorly constrained for carrying capacities larger than $10^5$ and for $\log_{10} \beta/\alpha$  greater than 2.5. This poor constraint is due primarily to the insensitivity of the abundance distribution to increasing values of these parameters. For example, moving to the far right side of the abundance phase diagram in Fig. 5B, the contours become flat in $\beta/\alpha$. 

To further constrain our estimates, we place upper bounds on these parameters with simple estimates of physical limits. To bound the carrying capacity, we note that a larval zebrafish intestine have a volume of roughly 1 nl, or $10^6$ $\mu$m$^3$. Taking the volume of a bacterium to be roughly 1 $\mu$m$^3$, we estimate a maximum bacterial load of 10$^6$ cells, consistent with the largest \textit{Vibrio} abundances (Fig. S3F). As we find no \textit{Enterobacter} populations above 10$^{5.5}$, and in our simulations we draw carrying capacities from a log-normal distribution with a standard deviation of half a decade, we constrained our best fit estimate to $\log_{10}K = 5.0$. To bound the fragmentation rate, $\beta$, we considered the time-lapse movie that showcases the greatest degree of cluster fragmentation observed (Supplemental Movie 9). This movie depicts the initial growth phase, in which both the size of aggregates and the number of single cells increase. Because the aggregates visibly grow in size, we know that the fragmentation rate must be bounded by the absolute growth rate of the population, $\beta < rN$; if the fragmentation rate were larger, cells would break off of the aggregate faster than they would be produced by cell division, and the aggregates would shrink in size. Taking, roughly, $r\sim 10^{-1}$ and $N\sim 10^3$ (Fig. 4D), we estimate that $\beta < 10^2$, or, with $\alpha = 10^{-1}$, $\beta/\alpha < 10^3$. With this bound, we constrain our best fit estimate to $\log_{10}\beta/\alpha = 2.5$. We took the uncertainties of the best fit estimates, $\sigma_{\log_{10}K}$ and $\sigma_{\log_{10}\beta/\alpha}$, to be the inverse of the local curvatures of $\chi^2$ at the best fit values: $\sigma_{\theta} = 1/|\partial^2_{\theta}\chi^2|$, for $\theta = \log_{10}K$, $\log_{10}\beta/\alpha$, resulting in $\sigma_{\log_{10}K} = 0.5$ and $\sigma_{\log_{10}K} = 0.4$.

\textit{Ciprofloxacin-treated Enterobacter}:  To estimate the change in \textit{Enterobacter}'s parameters upon antibiotic treatment, we conservatively assumed equal effects on growth and fragmentation/aggregation and modeled treatment parameters as $r' = \epsilon r$ and $\beta' = \epsilon \beta$. We then performed a single parameter grid search of $\epsilon$ values, ranging from $10^{-1.75}$ to $10^{-0.5}$. We modeled the antibiotic treatment as a parameter quench with a 6 hour buffer time, in which the antibiotics entered the intestine and began to take action on the bacteria. The value of 6 hours was chosen based on the \textit{Vibrio} time series data. Each simulation was initialized with a cluster configuration drawn randomly from the imaging-derived dataset of actual untreated \textit{Enterobacter} populations. The parameters $r$, $\lambda$, and $K$ were set to their best fit or measured values, $\alpha$ was again fixed at 0.1 hr$^{-1}$, and  $r$ and $\beta$ were both scaled by the same factors of $\epsilon$. We then ran simulations for a modified simulation time $24 - 6 = 18$ hours and fit the mean and standard deviation of shifted log-transformed abundances measured in the 24 hour treatment plating assays. A plot of $\chi^2$ vs $\epsilon$ shows a clear minimum at $\epsilon = 10^{-1}$ (Fig. S11D).

\textit{Untreated Vibrio}: Untreated \textit{Vibrio} populations are comprised of almost entirely single cells and therefore represent an extreme limit of the kinetic model. In this regime, fragmentation is so thorough that even dividing cells immediately separate and there is no appreciable aggregation. Because multicellular clusters are extremely rare, our data are insufficient to extract numerical estimates of model parameters. However, one can estimate a lower bound for the fragmentation rate, $\beta$, by equating it to the total growth rate, $rN$, where $N$ is the total population size; i.e. clusters do not grow without fragmenting. This estimate yields $\beta \gtrsim 10^5$. For the expulsion rate, if we assume the same rate as \textit{Enterobacter} (positing unchanged intestinal mechanics), we obtain $r/\lambda \sim 7$ These values place untreated \textit{Vibrio} off-scale in the phase diagram of Fig. 5B.

\textit{Ciprofloxacin-treated Vibrio}: We performed a two-parameter fit to ($\beta/\alpha$, $r$), using the measured expulsion rate for \textit{Enterobacter} ($\lambda = 0.11$ h$^{-1}$ and the typical untreated \textit{Vibrio} abundance for a carrying capacity of $K \sim 10^5$. We observed that in approaching the extinction transition from above, simulated abundance distributions transition from unimodal to bimodal in shape, with a peak emerging near $N=0$ representing populations that suffered large, abrupt collapses. As such, fitting just the mean and variance as was done for \textit{Enterobacter} produced inaccurate estimates. Therefore, we implemented full maximum likelihood estimation using 100 simulated replicates to estimate the likelihood. While the fit to treated \textit{Vibrio} resulted in less-constrained parameter estimates in the $r-\beta$ plane compared to the Enterobacter fit, it did yield a clear maximum (Fig. S12C) and a best-fit abundance distribution that matched experimental data within uncertainties (Fig. S12D). Like with \textit{Enterobacter}, we can attempt to assess the validity of this model by comparing the now-parameter-free prediction of the cluster size distribution with the image-derived data. Due to the rarity of large clusters and to limited data, the experimental distribution is severely undersampled. It shows, however, qualitative agreement with the model prediction (Fig. S12E). Finally, to confirm that our choice of the simulation timestep $\tau$ did not affect our parameter estimation, we decreased $\tau$ by a factor of 2 from $0.001$ h to $0.0005$ h and found no change in the best-fit cluster size distribution within sampling uncertainties (Fig. S12F). Because our parameter grid used in the fit was coarse, we estimate the uncertainty of our best-fit parameters as the grid spacing. Our uncertainty values are therefore likely overestimated.


%%%%% sup fig captions
%
%\newpage
%\section*{Figure S1 caption}
%\textbf{Measurement of \textit{Enterobacter} growth rate.} Image-derived quantification of initial growth dynamics in three zebrafish hosts. Imaging began approximately 8 hours after inoculation. 
%
%\section*{Figure S2 caption}
%\textbf{In vitro characterization of \textit{Vibrio} response to ciprofloxacin}. A: In vitro growth curves of \textit{Vibrio} in rich media (lysogeny broth) with different ciprofloxacin concentrations. B-C: Effects of ciprofloxacin on \textit{Vibrio} cell length and speed, with grey indicating experiments without antibiotic treatment and blue indicating exposure to 10 ng/ml ciprofloxacin. B: Distribution of \textit{Vibrio} cell lengths. Insets show representative fluorescence microscopy images of untreated and 10 ng/ml ciprofloxacin-treated cells; inset heights = 3.5 $\mu$m. C: Distribution of in vitro swimming speeds of individual bacteria.
%
%\section*{Figure S3 caption}
%\textbf{Additional \textit{Vibrio} data}  A: Representative masks of fluorescence microscopy images of in vitro viability staining. Top row, untreated, bottom row, 10 ng/ml ciprofloxacin-treated cells (6 hour treatment). SYTO9, shown in green (left panel), indicates intact cells, propridium iodine (PI), shown in magenta (middle panel), indicates dead cells. Double positive cells indicate damaged but viable cells \cite{ben2005genetic}, shown in white in the merged, right panel. Scale bar = 100 $\mu$m. B: Quantification of in vitro viability staining by fraction of cells corresponding to each case. Mean and standard deviation across 2 replicates shown. C: Representative fluorescence microscopy images of the SOS response in untreated (top row) and 10 ng/ml ciprofloxacin treated (bottom) cells. Constitutive dTom expression is shown in magenta (left), \textit{recN}-linked GFP expression in green (middle), merged images shown in right panel. Scale bar = 50 $\mu$m. D: Quantification of SOS response in fraction of SOS+ cells (Materials and Methods), mean and standard deviations shown, $n>4$ per treatment, total number of bacteria $>$ 120 cells per treatment. E: Timeline of in vivo antibiotic treatment. F: In vivo abundances of untreated and 10 ng/ml ciprofloxacin-treated cohorts by day. Each small circle corresponds to a single host, black lines indicate medians and quartiles. 
%
%\section*{Figure S4 caption}
%\textbf{Viability staining of \textit{Vibrio} cells expelled from the gut shows that ciprofloxacin does not induce widespread bacterial death in vivo}. Three examples of fish stained with SYTO9, which indicates live bacteria, and propidium iodide (PI), which indicates dead bacteria, for both untreated (A) and 10 ng/ml ciprofloxacin-treated (B) \textit{Vibrio}. Images were obtained by light sheet fluorescence microscopy and are maximum intensity projections of 3D images stacks. The field of view is around the vent region, as shown in the fish schematic at the top of the figure. The approximate boundary of the fish is indicated by the dashed orange line. Zebrafish cells also stain and constitute the bulk of the fluorescence in the images. Examples of zebrafish cells are indicated by white arrow heads. Examples of bacterial cells are indicated by the cyan arrows.
%
%
%\section*{Figure S5 caption}
%\textbf{In vivo ciprofloxacin dose response for \textit{Vibrio}.} \textit{Vibrio} was mono-associated with germ-free larval zebrafish for 24 hours prior to being left untreated, or treated with either 1, 10, or 100 ng/ml ciprofloxacin for an additional 24 hours.  \textit{Vibrio} abundances were determined by dissection and plating. Each circle corresponds to a single host intestine, black lines indicate medians and quartiles. Data for the `untreated' and `cipro 10 ng/ml' groups were included in Figure 2D and Supplemental Figure 2F, where they were combined with repeated experiments. 
%
%\section*{Figure S6 caption}
%\textbf{\textit{Vibrio} does not form large aggregates in vitro in response to ciprofloxacin}. Representative fluorescence microscopy images of untreated (A) and 10 ng/ml ciprofloxacin-treated (B) \textit{Vibrio} cells. Sample preparation and treatment are described in the \textit{Cell length and swimming speed} portion of the Materials and Methods section.
%
%\section*{Figure S7 caption}
%\textbf{Design and construction of fluorescent SOS reporter} A: Alignment of 100bp \textit{recN} promoter region plus start codon for the closely related \textit{V. cholerae} strains ZWU0020 (zebrafish isolate used in this study, IMG gene ID: 2705597027, locus tag: ZWU0020\_01601), ZOR0036 (zebrafish isolate, IMG gene ID: 2705599600, locus tag: ZOR0036\_00266), and El Tor N16961 (human pandemic isolate, IMG gene ID: 637047325, locus tag: VC0852). B: Alignment of 100bp \textit{recN} promoter region plus start codon of the \textit{V. cholerae} consensus \textit{recN} promoter, \textit{Aeromonas veronii} (zebrafish isolate, IMG gene ID: 2526373590, locus tag: L972\_03073), and \textit{E. coli} HS (human commensal isolate IMG gene ID: 640921890, locus tag: EcHS\_A2774). For panels A and B, SOS boxes are shaded based on their conservation to the consensus gammaproteobacterial sequence (\texttt{CTGTN$_8$ACAG}); `ATG' start codons are bolded; putative ribosome binding sites are boxed; and putative, near-consensus -10 promoter sequences (\texttt{TATAAT}) are bolded and underlined. C: Schematic of \textit{recN}-based fluorescent SOS reporter. Promoter comprises the consensus \textit{V. cholerae} \textit{recN} promoter region (P\textit{recN}), which was derived from the sequence alignment in panel A. The synthetic 5' untranslated region (UTR) contains an epsilon enhancer and consensus Shine-Dalgarno sequence. The open reading frame (ORF) encodes superfolder green fluorescent protein (sfGFP). And the 3' UTR contains the synthetic transcriptional terminator L3S2P21. D: Schematic of assembled SOS reporter in the context of the Tn\textit{7} tagging construct. Tn\textit{7}L and Tn\textit{7}R inverted repeats flank the Tn\textit{7} transposon. The SOS reporter was inserted upstream of a dTomato gene that is constitutively expressed from a synthetic Ptac promoter. A gene encoding gentamicin resistance (\textit{gentR}) was used to facilitate genetic manipulation. E: Disk diffusion assays verifying SOS reporter activity. \textit{Vibrio} ZWU0020 carrying the SOS reporter was spread onto agar plates using glass beads at a density that would give rise to a lawn of growth. Circular disks of Whatman filter paper (amber dashed lines) loaded with either the genotoxic agent mitomycin C or the cell wall-targeting beta-lactam antibiotic ampicillin were then placed in the center of the agar plates. After overnight incubation at 30$^{\circ}$C, plates were imaged using a fluorescence stereomicroscope. In the presence of mitomycin C, cells adjacent to the zone of inhibition (i.e., the area where there is no bacterial growth) robustly expressed sfGFP whereas in the presence of ampicillin they did not.    
%
%
%\section*{Figure S8 caption}
%In vitro growth curves (in lysogeny broth) of \textit{Enterobacter} with varying concentrations of ciprofloxacin.
%
%\section*{Figure S9 caption}
%\textbf{Additional \textit{Enterobacter} data}  A: Representative fluorescence microscopy images of in vitro viability staining. Top row, untreated, bottom row, 25 ng/ml ciprofloxacin-treated cells (6 hour treatment). SYTO9, shown in green (left panel), indicates intact cells, propridium iodine (PI), shown in magenta (middle panel), indicates dead cells. Double positive cells indicate damaged but viable cells \cite{ben2005genetic}, shown in white in the merged, right panel. Scale bar = 100 $\mu$m. B: Quantification of in vitro viability staining by fraction of cells corresponding to each case. Mean and standard deviation across 2 replicates shown. C: Timeline of in vivo antibiotic treatment. D: In vivo abundances of untreated and 25 ng/ml ciprofloxacin-treated cohorts by day. Each small circle corresponds to a single host, black lines indicate medians and quartiles. E: Maximum intensity projection of untreated \textit{Enterobacter} population showing an example of a population containing a single large cluster and several single cells. Scale bar = 200 $\mu$m.
%
%\section*{Figure S10 caption}
%\textbf{Viability staining of \textit{Enterobacter} cells expelled from the gut shows that ciprofloxacin does not induce widespread bacterial death in vivo}. Three examples of fish stained with SYTO9, which indicates live bacteria, and propidium iodide (PI), which indicates dead bacteria, for both untreated (A) and 25 ng/ml ciprofloxacin-treated (B) \textit{Enterobacter}. Images were obtained by light sheet fluorescence microscopy and are maximum intensity projections of 3D images stacks. The field of view is around the vent region, as shown in the fish schematic at the top of the figure. The approximate boundary of the fish is indicated by the dashed orange line. Zebrafish cells also stain and constitute the bulk of the fluorescence in the images. Examples of zebrafish cells are indicated by white arrow heads. Examples of bacterial cells are indicated by the cyan arrows.
%
%\section*{Figure S11 caption}
%\textbf{Additional model details.}  A-B: Simulated heatmap of mean (A) and standard deviation (B) of $\log_{10}(\text{abundance} + 1)$ for varying values of aggregation and fragmentation rates. Both mean and standard deviation depend primarily on the ratio of fragmentation to aggregation rates, rather than on each rate independently. Dashed magenta line in (A) represents $\alpha = \beta$. Parameters: $r = 0.27$ hr$^{-1}$, $\lambda = 0.11$ hr$^{-1}$, $K = 10^5$, simulation time $= 64$ hours, number of trials decreased logarithmically with $\beta$ from 1000 to 10. Units of $\alpha$ and $\beta$ are hr$^{-1}$.  C: Heatmap of $\chi^2$ for untreated \textit{Enterobacter} fit to 7 dpf abundances (Materials and Methods). Parameters: $r = 0.27$ hr$^{-1}$, $\lambda = 0.11$ hr$^{-1}$, $\alpha = 0.1$ hr$^{-1}$, simulation time $= 64$ hours, number of trials decreased logarithmically with $\beta$ from 1000 to 10. D:   $\chi^2$ for fit to 6 dpf ciprofloxacin-treated \textit{Enterobacter} abundances as a function of the scaling parameter $\epsilon$, which scales the growth and fragmentation rates simultaneously according to $r \to \epsilon r$ and $\beta \to \epsilon \beta$. A clear minimum is seen at $\epsilon = 0.1$. Parameters: $r = 0.27$ hr$^{-1}$, $\lambda = 0.11$ hr$^{-1}$, $\alpha = 0.1$ hr$^{-1}$, $\beta = 10^{1.5}$ hr$^{-1}$, simulation time $= 64$ hours, number of trials decreased logarithmically with $\beta$ from 1000 to 10. E: 3D phase diagram of $\log_{10}(\text{abundance} + 1)$ with axes fragmentation/aggregation ($\beta/\alpha$), growth rate ($r$), and expulsion rate ($\lambda$). Blue isosurface represents $\log_{10}(\text{abundance} + 1) = 0.5 \pm 0.5$, yellow isosurface represents  $\log_{10}(\text{abundance} + 1) = 5.5 \pm 0.5$. Parameters: $\alpha = 0.1$ hr$^{-1}$, simulation time $= 64$ hours, number of trials decreased logarithmically with $\beta$ from 1000 to 10. Units of $\alpha$ and $\beta$ are hr$^{-1}$. F: Slices through the 3D phase diagram in (E) for different values of $\lambda$.
%
%\section*{Figure S12 caption}
%\textbf{Tau leaping simulations and \textit{Vibrio} parameter inference}. A-B: Comparison of direct stochastic simulation (``ssa'', gray circles) and our fixed-tau leaping (``tau'', purple diamonds) algorithm with $\tau = 0.001$ h. Simulations using both methods were run with the best-fit parameters for untreated \textit{Enterobacter} and 100 replicates. Both the cluster size distribution (A) and abundance histogram (B) show excellent agreement between the two methods. C-E: Details of model fit to ciprofloxacin-treated \textit{Vibrio} 24 h abundances. C: Heat map of log-likelihood. A manual grid search was performed over growth rate ($r$) and fragmentation rate ($\beta$). D: Comparison of the best-fit abundance distribution (purple line) to experimental data (blue circles). E: Comparison of the predicted cluster size distribution (purple line) to experimental data (blue circles). Here, all model parameters were fixed at their previously determined, best-fit values; there were no additional free parameters. The experimental data distribution is severely undersampled, estimated from just 4 fish. F: Confirmation that the best-fit solution is independent of our choice of $\tau$, indicating that simulations were performed with sufficient resolution. Simulations were run with the best-fit parameters but with $\tau$ decreased by a factor of 2, from $\tau=0.001$ h (purple circles) to $\tau = 0.0005$ h (green diamonds). Distributions agree with one another within sampling uncertainties.
%
%\section*{Figure S13 caption}
%\textbf{Model cluster size distributions are independent of how expulsion rate scales with cluster size}. Simulations were run with the expulsion rate depending on cluster size according to $E_n = \lambda (n/K)^{\nu}$, with $K$ the carrying capacity, and the exponent $\nu$ was varied. This ansatz keeps the expulsion rate of clusters of size $K$ constant. The resulting cluster size distributions agree with one another within sampling uncertainties, which are smaller than the marker size. This result justifies our use of the simple constant form of the expulsion kernel, $E_n = \lambda$.
%
%\newpage
\renewcommand\thefigure{S\arabic{figure}}    
%\renewcommand{\figurename}{Supplementary Figure }


\setcounter{figure}{0}

\section{Supplementary Figures}
%\section*{Supplementary Figure 1}
\begin{figure}[H]
	\centerline{
		\includegraphics[width =3 in]{../../dissertation_figs/abx/figS1.eps}}
	\caption{\textbf{Measurement of \textit{Enterobacter} growth rate.}}{Image-derived quantification of initial growth dynamics in three zebrafish hosts. Imaging began approximately 8 hours after inoculation.}
\end{figure}



%\newpage
%\section*{Supplementary Figure 2}

\begin{figure}[H]
	\centerline{
		\includegraphics[width =6 in]{../../dissertation_figs/abx/figS2.eps}}
	\caption{\textbf{In vitro characterization of \textit{Vibrio} response to ciprofloxacin}.}{A: In vitro growth curves of \textit{Vibrio} in rich media (lysogeny broth) with different ciprofloxacin concentrations. B-C: Effects of ciprofloxacin on \textit{Vibrio} cell length and speed, with grey indicating experiments without antibiotic treatment and blue indicating exposure to 10 ng/ml ciprofloxacin. B: Distribution of \textit{Vibrio} cell lengths. Insets show representative fluorescence microscopy images of untreated and 10 ng/ml ciprofloxacin-treated cells; inset heights = 3.5 $\mu$m. C: Distribution of in vitro swimming speeds of individual bacteria.}
\end{figure}


	 
%\newpage
%\section*{Supplementary Figure 3}

\begin{figure}[H]
	\centerline{
		\includegraphics[width =6 in]{../../dissertation_figs/abx/figS3.eps}}
	\caption{\textbf{Additional \textit{Vibrio} data}}{(Continued in footnote.)}
	\label{fig:figS3}
\end{figure}

{\let\thefootnote\relax\footnote{FIGURE \ref{fig:figS3} (cont'd) A: Representative masks of fluorescence microscopy images of in vitro viability staining. Top row, untreated, bottom row, 10 ng/ml ciprofloxacin-treated cells (6 hour treatment). SYTO9, shown in green (left panel), indicates intact cells, propridium iodine (PI), shown in magenta (middle panel), indicates dead cells. Double positive cells indicate damaged but viable cells \cite{ben2005genetic}, shown in white in the merged, right panel. Scale bar = 100 $\mu$m. B: Quantification of in vitro viability staining by fraction of cells corresponding to each case. Mean and standard deviation across 2 replicates shown. C: Representative fluorescence microscopy images of the SOS response in untreated (top row) and 10 ng/ml ciprofloxacin treated (bottom) cells. Constitutive dTom expression is shown in magenta (left), \textit{recN}-linked GFP expression in green (middle), merged images shown in right panel. Scale bar = 50 $\mu$m. D: Quantification of SOS response in fraction of SOS+ cells (Materials and Methods), mean and standard deviations shown, $n>4$ per treatment, total number of bacteria $>$ 120 cells per treatment. E: Timeline of in vivo antibiotic treatment. F: In vivo abundances of untreated and 10 ng/ml ciprofloxacin-treated cohorts by day. Each small circle corresponds to a single host, black lines indicate medians and quartiles.}

%\newpage
%\section*{Supplementary Figure 4}

\begin{figure}[H]
	\centerline{
		\includegraphics[width =4.5 in]{../../dissertation_figs/abx/figS4.eps}}
	\caption{\textbf{Viability staining of \textit{Vibrio} cells expelled from the gut shows that ciprofloxacin does not induce widespread bacterial death in vivo}.}{(Continued in footnote.)}
	\label{fig:figS4}
\end{figure}

{\let\thefootnote\relax\footnote{FIGURE \ref{fig:figS4} (cont'd): Three examples of fish stained with SYTO9, which indicates live bacteria, and propidium iodide (PI), which indicates dead bacteria, for both untreated (A) and 10 ng/ml ciprofloxacin-treated (B) \textit{Vibrio}. Images were obtained by light sheet fluorescence microscopy and are maximum intensity projections of 3D images stacks. The field of view is around the vent region, as shown in the fish schematic at the top of the figure. The approximate boundary of the fish is indicated by the dashed orange line. Zebrafish cells also stain and constitute the bulk of the fluorescence in the images. Examples of zebrafish cells are indicated by white arrow heads. Examples of bacterial cells are indicated by the cyan arrows.}
		
%\newpage
%\section*{Supplementary Figure 5}

\begin{figure}[H]
	\centerline{
		\includegraphics[width =3 in]{../../dissertation_figs/abx/figS5.eps}}
	\caption{\textbf{In vivo ciprofloxacin dose response for \textit{Vibrio}:}}{\textit{Vibrio} was mono-associated with germ-free larval zebrafish for 24 hours prior to being left untreated, or treated with either 1, 10, or 100 ng/ml ciprofloxacin for an additional 24 hours.  \textit{Vibrio} abundances were determined by dissection and plating. Each circle corresponds to a single host intestine, black lines indicate medians and quartiles. Data for the `untreated' and `cipro 10 ng/ml' groups were included in Figure 2D and Supplemental Figure 2F, where they were combined with repeated experiments.}
\end{figure}


%\newpage
%\section*{Supplementary Figure 6}

\begin{figure}[H]
	\centerline{
		\includegraphics[width =6 in]{../../dissertation_figs/abx/figS6.eps}} 
	\caption{\textbf{\textit{Vibrio} does not form large aggregates in vitro in response to ciprofloxacin}.}
	\label{fig:figS6}
\end{figure}

{\let\thefootnote\relax\footnote{FIGURE \ref{fig:figS6} (cont'd) Representative fluorescence microscopy images of untreated (A) and 10 ng/ml ciprofloxacin-treated (B) \textit{Vibrio} cells. Sample preparation and treatment are described in the \textit{Cell length and swimming speed} portion of the Materials and Methods section.}
		
%\newpage
%\section*{Supplementary Figure 7}

\begin{figure}[H]
	\centerline{
		\includegraphics[width =6 in]{../../dissertation_figs/abx/figS7.eps}} 
	\caption{\textbf{Design and construction of fluorescent SOS reporter.}}
	\label{fig:figS7}
\end{figure}
{\let\thefootnote\relax\footnote{FIGURE \ref{fig:figS7} (cont'd) A: Alignment of 100bp \textit{recN} promoter region plus start codon for the closely related \textit{V. cholerae} strains ZWU0020 (zebrafish isolate used in this study, IMG gene ID: 2705597027, locus tag: ZWU0020\_01601), ZOR0036 (zebrafish isolate, IMG gene ID: 2705599600, locus tag: ZOR0036\_00266), and El Tor N16961 (human pandemic isolate, IMG gene ID: 637047325, locus tag: VC0852). B: Alignment of 100bp \textit{recN} promoter region plus start codon of the \textit{V. cholerae} consensus \textit{recN} promoter, \textit{Aeromonas veronii} (zebrafish isolate, IMG gene ID: 2526373590, locus tag: L972\_03073), and \textit{E. coli} HS (human commensal isolate IMG gene ID: 640921890, locus tag: EcHS\_A2774). For panels A and B, SOS boxes are shaded based on their conservation to the consensus gammaproteobacterial sequence (\texttt{CTGTN$_8$ACAG}); `ATG' start codons are bolded; putative ribosome binding sites are boxed; and putative, near-consensus -10 promoter sequences (\texttt{TATAAT}) are bolded and underlined. C: Schematic of \textit{recN}-based fluorescent SOS reporter. Promoter comprises the consensus \textit{V. cholerae} \textit{recN} promoter region (P\textit{recN}), which was derived from the sequence alignment in panel A. The synthetic 5' untranslated region (UTR) contains an epsilon enhancer and consensus Shine-Dalgarno sequence. The open reading frame (ORF) encodes superfolder green fluorescent protein (sfGFP). And the 3' UTR contains the synthetic transcriptional terminator L3S2P21. D: Schematic of assembled SOS reporter in the context of the Tn\textit{7} tagging construct. Tn\textit{7}L and Tn\textit{7}R inverted repeats flank the Tn\textit{7} transposon. The SOS reporter was inserted upstream of a dTomato gene that is constitutively expressed from a synthetic Ptac promoter. A gene encoding gentamicin resistance (\textit{gentR}) was used to facilitate genetic manipulation. E: Disk diffusion assays verifying SOS reporter activity. \textit{Vibrio} ZWU0020 carrying the SOS reporter was spread onto agar plates using glass beads at a density that would give rise to a lawn of growth. Circular disks of Whatman filter paper (amber dashed lines) loaded with either the genotoxic agent mitomycin C or the cell wall-targeting beta-lactam antibiotic ampicillin were then placed in the center of the agar plates. After overnight incubation at 30$^{\circ}$C, plates were imaged using a fluorescence stereomicroscope. In the presence of mitomycin C, cells adjacent to the zone of inhibition (i.e., the area where there is no bacterial growth) robustly expressed sfGFP whereas in the presence of ampicillin they did not.}
%\newpage
%\section*{Supplementary Figure 8}

\begin{figure}[H]
	\centerline{
		\includegraphics[width =3 in]{../../dissertation_figs/abx/figS8.eps}} 
	\caption{\textbf{In vitro growth curves (in lysogeny broth) of \textit{Enterobacter} with varying concentrations of ciprofloxacin.}}
\end{figure}

%\newpage
%\section*{Supplementary Figure 9}

\begin{figure}[H]
	\centerline{
		\includegraphics[width =6 in]{../../dissertation_figs/abx/figS9.eps}}
	\caption{\textbf{Additional \textit{Enterobacter} data.}} 
	\label{fig:figS9}
\end{figure}

{\let\thefootnote\relax\footnote{FIGURE \ref{fig:figS9} (cont'd) A: Representative fluorescence microscopy images of in vitro viability staining. Top row, untreated, bottom row, 25 ng/ml ciprofloxacin-treated cells (6 hour treatment). SYTO9, shown in green (left panel), indicates intact cells, propridium iodine (PI), shown in magenta (middle panel), indicates dead cells. Double positive cells indicate damaged but viable cells \cite{ben2005genetic}, shown in white in the merged, right panel. Scale bar = 100 $\mu$m. B: Quantification of in vitro viability staining by fraction of cells corresponding to each case. Mean and standard deviation across 2 replicates shown. C: Timeline of in vivo antibiotic treatment. D: In vivo abundances of untreated and 25 ng/ml ciprofloxacin-treated cohorts by day. Each small circle corresponds to a single host, black lines indicate medians and quartiles. E: Maximum intensity projection of untreated \textit{Enterobacter} population showing an example of a population containing a single large cluster and several single cells. Scale bar = 200 $\mu$m.}

%\newpage
%\section*{Supplementary Figure 10}

\begin{figure}[H]
	\centerline{
		\includegraphics[width =4 in]{../../dissertation_figs/abx/figS10.eps}} 
	\caption{\textbf{Viability staining of \textit{Enterobacter} cells expelled from the gut shows that ciprofloxacin does not induce widespread bacterial death in vivo}.}
	\label{fig:figS10}
\end{figure}

{\let\thefootnote\relax\footnote{FIGURE \ref{fig:figS10} (cont'd) Three examples of fish stained with SYTO9, which indicates live bacteria, and propidium iodide (PI), which indicates dead bacteria, for both untreated (A) and 25 ng/ml ciprofloxacin-treated (B) \textit{Enterobacter}. Images were obtained by light sheet fluorescence microscopy and are maximum intensity projections of 3D images stacks. The field of view is around the vent region, as shown in the fish schematic at the top of the figure. The approximate boundary of the fish is indicated by the dashed orange line. Zebrafish cells also stain and constitute the bulk of the fluorescence in the images. Examples of zebrafish cells are indicated by white arrow heads. Examples of bacterial cells are indicated by the cyan arrows.}
	
%\newpage
%\section*{Supplementary Figure 11}

\begin{figure}[H]
	\centerline{
		\includegraphics[width =6 in]{../../dissertation_figs/abx/figS11.eps}} 
	\caption{\textbf{Additional model details}.}
	\label{fig:figS11}
\end{figure}

{\let\thefootnote\relax\footnote{FIGURE \ref{fig:figS11} (cont'd) A-B: Simulated heatmap of mean (A) and standard deviation (B) of $\log_{10}(\text{abundance} + 1)$ for varying values of aggregation and fragmentation rates. Both mean and standard deviation depend primarily on the ratio of fragmentation to aggregation rates, rather than on each rate independently. Dashed magenta line in (A) represents $\alpha = \beta$. Parameters: $r = 0.27$ hr$^{-1}$, $\lambda = 0.11$ hr$^{-1}$, $K = 10^5$, simulation time $= 64$ hours, number of trials decreased logarithmically with $\beta$ from 1000 to 10. Units of $\alpha$ and $\beta$ are hr$^{-1}$.  C: Heatmap of $\chi^2$ for untreated \textit{Enterobacter} fit to 7 dpf abundances (Materials and Methods). Parameters: $r = 0.27$ hr$^{-1}$, $\lambda = 0.11$ hr$^{-1}$, $\alpha = 0.1$ hr$^{-1}$, simulation time $= 64$ hours, number of trials decreased logarithmically with $\beta$ from 1000 to 10. D:   $\chi^2$ for fit to 6 dpf ciprofloxacin-treated \textit{Enterobacter} abundances as a function of the scaling parameter $\epsilon$, which scales the growth and fragmentation rates simultaneously according to $r \to \epsilon r$ and $\beta \to \epsilon \beta$. A clear minimum is seen at $\epsilon = 0.1$. Parameters: $r = 0.27$ hr$^{-1}$, $\lambda = 0.11$ hr$^{-1}$, $\alpha = 0.1$ hr$^{-1}$, $\beta = 10^{1.5}$ hr$^{-1}$, simulation time $= 64$ hours, number of trials decreased logarithmically with $\beta$ from 1000 to 10. E: 3D phase diagram of $\log_{10}(\text{abundance} + 1)$ with axes fragmentation/aggregation ($\beta/\alpha$), growth rate ($r$), and expulsion rate ($\lambda$). Blue isosurface represents $\log_{10}(\text{abundance} + 1) = 0.5 \pm 0.5$, yellow isosurface represents  $\log_{10}(\text{abundance} + 1) = 5.5 \pm 0.5$. Parameters: $\alpha = 0.1$ hr$^{-1}$, simulation time $= 64$ hours, number of trials decreased logarithmically with $\beta$ from 1000 to 10. Units of $\alpha$ and $\beta$ are hr$^{-1}$. F: Slices through the 3D phase diagram in (E) for different values of $\lambda$.}

%\newpage
%\section*{Supplementary Figure 12}

\begin{figure}[H]
	\centerline{
		\includegraphics[width =6 in]{../../dissertation_figs/abx/figS12.eps}} 
	\caption{\textbf{Tau leaping simulations and \textit{Vibrio} parameter inference}.}
	\label{fig:figS12}
\end{figure}

{\let\thefootnote\relax\footnote{FIGURE \ref{fig:figS12} (cont'd) A-B: Comparison of direct stochastic simulation (``ssa'', gray circles) and our fixed-tau leaping (``tau'', purple diamonds) algorithm with $\tau = 0.001$ h. Simulations using both methods were run with the best-fit parameters for untreated \textit{Enterobacter} and 100 replicates. Both the cluster size distribution (A) and abundance histogram (B) show excellent agreement between the two methods. C-E: Details of model fit to ciprofloxacin-treated \textit{Vibrio} 24 h abundances. C: Heat map of log-likelihood. A manual grid search was performed over growth rate ($r$) and fragmentation rate ($\beta$). D: Comparison of the best-fit abundance distribution (purple line) to experimental data (blue circles). E: Comparison of the predicted cluster size distribution (purple line) to experimental data (blue circles). Here, all model parameters were fixed at their previously determined, best-fit values; there were no additional free parameters. The experimental data distribution is severely undersampled, estimated from just 4 fish. F: Confirmation that the best-fit solution is independent of our choice of $\tau$, indicating that simulations were performed with sufficient resolution. Simulations were run with the best-fit parameters but with $\tau$ decreased by a factor of 2, from $\tau=0.001$ h (purple circles) to $\tau = 0.0005$ h (green diamonds). Distributions agree with one another within sampling uncertainties.}

%\newpage
%\section*{Supplementary Figure 13}

\begin{figure}[H]
	\centerline{
		\includegraphics[width =3 in]{../../dissertation_figs/abx/figS13.eps}} 
	\caption{\textbf{Model cluster size distributions are independent of how expulsion rate scales with cluster size}.}
	\label{fig:figS13}
\end{figure}

{\let\thefootnote\relax\footnote{FIGURE \ref{fig:figS13} (cont'd) Simulations were run with the expulsion rate depending on cluster size according to $E_n = \lambda (n/K)^{\nu}$, with $K$ the carrying capacity, and the exponent $\nu$ was varied. This ansatz keeps the expulsion rate of clusters of size $K$ constant. The resulting cluster size distributions agree with one another within sampling uncertainties, which are smaller than the marker size. This result justifies our use of the simple constant form of the expulsion kernel, $E_n = \lambda$.}

\renewcommand\thefigure{\arabic{figure}}    
\newpage

\section{Supplemental Movie captions}

\subsection*{Supplemental Movie 1} 
Light sheet fluorescence microscopy movie of untreated \textit{Vibrio} swimming in a 6 dpf zebrafish gut. The density of cells is highest on the left (anterior), where single cells cannot be resolved and the population appears as a single bright region (see also Figure 1C). On the right (posterior), single cells are more easily resolved and are seen swimming in and out of the intestinal folds. Each frame is from the same optical plane. Scale bar = 50 $\mu$m.

\subsection*{Supplemental Movie 2} 
Animated z-stack of light sheet fluorescence microscopy images of untreated \textit{Enterobacter} in a 6 dpf zebrafish gut. Bacterial clusters (bright white puncta) of diverse sizes are evident, from single cells up to a single cluster containing thousands of cells that appears at a z depth of $\sim$ 70 $\mu$m. Hazy reflection of light off of the fish's swim bladder can be seen outside the intestinal boundary in the upper right section of the images. Scale bar = 50 $\mu$m.

\subsection*{Supplemental Movie 3}
Fluorescence microscopy movie of untreated \textit{Vibrio} swimming between a glass slide and a coverslip (Materials and Methods). Scale bar = 20 $\mu$m.

\subsection*{Supplemental Movie 4}
Fluorescence microscopy movie of \textit{Vibrio} treated with 10 ng/ml ciprofloxacin swimming between a glass slide and a coverslip (Materials and Methods). Cells have undergone filamentation. Scale bar = 20 $\mu$m.

\subsection*{Supplemental Movie 5}
Time-lapse light sheet fluorescence microscopy movie of an established \textit{Vibrio} population responding to 10 ng/ml ciprofloxacin. Each frame is a maximum intensity projection of the full 3D intestinal volume.  The time between frames is 20 min. Initially, the population consists of a dense collection of individual, motile cells (Supplemental Movie 1, Figure 1C). Antibiotics are added after the second frame of the movie.  Following motility loss, cells leave the swarm and are compacted into aggregates, which are subject to strong transport down the length of the intestine and are eventually expelled. Scale bar = 200 $\mu$m.

\subsection*{Supplemental Movie 6}
Light sheet fluorescence microscopy movies of \textit{Vibrio} in fish treated with 10 ng/ml ciprofloxacin. The left panel movie shows constitutive dTom expression. The right panel movie was taken immediately after the left panel movie and shows a GFP reporter of the SOS response (Materials and Methods), which is expressed in cells strongly affected by ciprofloxacin (Fig. S3C and S3D). GFP-positive cells swim slowly or are aggregated. Each frame is from the same optical plane. Scale bar = 25 $\mu$m. 

\subsection*{Supplemental Movie 7}
Time-lapse light sheet fluorescence microscopy movie of an untreated \textit{Enterobacter} population showing an example of the expulsion process. Each frame is a maximum intensity projection of the full 3D intestinal volume. Time between frames is 10 min. The population is initially comprised of many small bacterial clusters and a single large cluster. Over time, small clusters are incorporated into the large one and the mass is transported down the length of the gut and expelled. Image intensities are log-transformed. Scale bar = 200 $\mu$m.

\subsection*{Supplemental Movie 8}
Time-lapse light sheet fluorescence microscopy movie of an untreated \textit{Enterobacter} population showing an example of the aggregation process. Each frame is a maximum intensity projection of the full 3D intestinal volume. Time between frames is 10 min. A collection of initially disconnected bacterial clusters on the left (anterior) side of the field of view gradually combine into a single cluster. Image intensities are log-transformed. Scale bar = 200 $\mu$m.

\subsection*{Supplemental Movie 9}
Time-lapse light sheet fluorescence microscopy movie of an untreated \textit{Enterobacter} population showing examples of the growth and fragmentation processes. Each frame is a maximum intensity projection of the full 3D intestinal volume. The time between frames is 20 min. The movie begins 8 hours after the initial exposure to \textit{Enterobacter}, by which time a small founding population has been established. Over time, the aggregates grow in size as cells divide, and new single cells also appear in the vicinity of the aggregates, likely due to fragmentation. Individual cell divisions from planktonic cells are also visible. Image intensities are log-transformed. Scale bar = 200 $\mu$m.

\subsection*{Supplemental Movie 10}
Light sheet fluorescence microscopy movie of \textit{Vibrio} in a fish treated with 10 ng/ml ciprofloxacin for $\sim$18 hours. Each frame is from the same optical plane, which spans the anterior-most region of the intestine known as the intestinal bulb (Fig. 1B). The bright signal in the left (anterior) side of the frame is a dense, motile swarm of planktonic cells (Supplemental Movie 1 and Fig. 1C). Moving from left to right (anterior-posterior) across the field of view, cells exhibiting filamentation and reduced motility are evident, along with the beginnings of small aggregates. Scale bar = 50 $\mu$m.

\subsection*{Supplemental Movie 11}
Light sheet fluorescence microsocopy movie of \textit{Vibrio} in a fish treated with 10 ng/ml ciprofloxacin for $\sim$18 hours. Each frame from the same single optical plane that captures a portion of the midgut (Fig. 1B). The bright signal is an aggregate of \textit{Vibrio} cells that nearly fills the width of the midgut lumen. Two cells are seen swimming near the end of the movie. Scale bar = 25 $\mu$m.






%\bibliography{/Users/brandonschlomann/Documents/Gutz/PaperWriting/abx/abx_refs.bib}{}
%\bibliographystyle{unsrt}
%\begin{thebibliography}{10}
%
%\bibitem{relmanABX_2011}
%Dethlefsen L, Relman DA (2011) Incomplete recovery and individualized responses
%  of the human distal gut microbiota to repeated antibiotic perturbation.
%\newblock {\em Proceedings of the National Academy of Sciences} 108(Supplement
%  1):4554--4561.
%
%\bibitem{cho2012antibiotics}
%Cho I, et~al. (2012) Antibiotics in early life alter the murine colonic
%  microbiome and adiposity.
%\newblock {\em Nature} 488(7413):621.
%
%\bibitem{schulfer2019impact}
%Schulfer AF, et~al. (2019, doi:10.1038/s41396-019-0349-4, published online
%  ahead of print) The impact of early-life sub-therapeutic antibiotic treatment
%  ({STAT}) on excessive weight is robust despite transfer of intestinal
%  microbes.
%\newblock {\em The ISME Journal}.
%
%\bibitem{gaulke2016triclosan}
%Gaulke CA, Barton CL, Proffitt S, Tanguay RL, Sharpton TJ (2016) Triclosan
%  exposure is associated with rapid restructuring of the microbiome in adult
%  zebrafish.
%\newblock {\em PLoS One} 11(5):e0154632.
%
%\bibitem{andersson2014microbiological}
%Andersson DI, Hughes D (2014) Microbiological effects of sublethal levels of
%  antibiotics.
%\newblock {\em Nature Reviews Microbiology} 12(7):465.
%
%\bibitem{national2018environmental}
%"National Academies~of Sciences E, Medicine, others" (2018) {\em Environmental
%  Chemicals, the Human Microbiome, and Health Risk: A Research Strategy}.
%\newblock (National Academies Press).
%
%\bibitem{walters2003contributions}
%Walters MC, Roe F, Bugnicourt A, Franklin MJ, Stewart PS (2003) Contributions
%  of antibiotic penetration, oxygen limitation, and low metabolic activity to
%  tolerance of pseudomonas aeruginosa biofilms to ciprofloxacin and tobramycin.
%\newblock {\em Antimicrobial Agents and Chemotherapy} 47(1):317--323.
%
%\bibitem{fux2005survival}
%Fux C, Costerton JW, Stewart PS, Stoodley P (2005) Survival strategies of
%  infectious biofilms.
%\newblock {\em Trends in Microbiology} 13(1):34--40.
%
%\bibitem{korem2015growth}
%Korem T, et~al. (2015) Growth dynamics of gut microbiota in health and disease
%  inferred from single metagenomic samples.
%\newblock {\em Science} 349(6252):1101--1106.
%
%\bibitem{schlomann_bacterial_2018}
%Schlomann BH, Wiles TJ, Wall ES, Guillemin K, Parthasarathy R (2018) Bacterial
%  cohesion predicts spatial distribution in the larval zebrafish intestine.
%\newblock {\em Biophysical Journal} 115(11):2271--2277.
%
%\bibitem{moor_high-avidity_2017}
%Moor K, et~al. (2017) {High-avidity IgA protects the intestine by enchaining
%  growing bacteria}.
%\newblock {\em Nature} 544(7651):498.
%
%\bibitem{welch_spatial_2017}
%Welch JLM, Hasegawa Y, McNulty NP, Gordon JI, Borisy GG (2017) Spatial
%  organization of a model 15-member human gut microbiota established in
%  gnotobiotic mice.
%\newblock {\em Proceedings of the National Academy of Sciences}
%  114(43):E9105--E9114.
%
%\bibitem{cremer2017effect}
%Cremer J, Arnoldini M, Hwa T (2017) Effect of water flow and chemical
%  environment on microbiota growth and composition in the human colon.
%\newblock {\em Proceedings of the National Academy of Sciences}
%  114(25):6438--6443.
%
%\bibitem{stephens_composition_2016}
%Stephens WZ, et~al. (2016) {The composition of the zebrafish intestinal
%  microbial community varies across development}.
%\newblock {\em ISME Journal} 10:644--654.
%
%\bibitem{wiles_host_2016}
%Wiles TJ, et~al. (2016) {Host Gut Motility Promotes Competitive Exclusion
%  within a Model Intestinal Microbiota}.
%\newblock {\em PLoS Biology} 14(7):1--24.
%
%\bibitem{wiles_modernized_2018}
%Wiles TJ, et~al. (2018) Modernized tools for streamlined genetic manipulation
%  and comparative study of wild and diverse proteobacterial lineages.
%\newblock {\em mBio} 9(5):e01877--18.
%
%\bibitem{relmanABX_2011}
%Dethlefsen L, Relman DA (2011) Incomplete recovery and individualized responses
%  of the human distal gut microbiota to repeated antibiotic perturbation.
%\newblock {\em Proceedings of the National Academy of Science} 108.
%
%\bibitem{girardi2011biodegradation}
%Girardi C, et~al. (2011) Biodegradation of ciprofloxacin in water and soil and
%  its effects on the microbial communities.
%\newblock {\em Journal of Hazardous Materials} 198:22--30.
%
%\bibitem{goneau2015subinhibitory}
%Goneau LW, et~al. (2015) Subinhibitory antibiotic therapy alters recurrent
%  urinary tract infection pathogenesis through modulation of bacterial
%  virulence and host immunity.
%\newblock {\em mBio} 6(2):e00356--15.
%
%\bibitem{rennekamp201515}
%Rennekamp AJ, Peterson RT (2015) 15 years of zebrafish chemical screening.
%\newblock {\em Current opinion in chemical biology} 24:58--70.
%
%\bibitem{yoganantharjah2017use}
%Yoganantharjah P, Gibert Y (2017) The use of the zebrafish model to aid in drug
%  discovery and target validation.
%\newblock {\em Current topics in medicinal chemistry} 17(18):2041--2055.
%
%\bibitem{erill2007aeons}
%Erill I, Campoy S, Barb{\'e} J (2007) Aeons of distress: an evolutionary
%  perspective on the bacterial sos response.
%\newblock {\em FEMS Microbiology Reviews} 31(6):637--656.
%
%\bibitem{kreuzer2013dna}
%Kreuzer KN (2013) Dna damage responses in prokaryotes: regulating gene
%  expression, modulating growth patterns, and manipulating replication forks.
%\newblock {\em Cold Spring Harbor Perspectives in Biology} 5(11):a012674.
%
%\bibitem{irazoki2016sos}
%Irazoki O, Mayola A, Campoy S, Barb{\'e} J (2016) Sos system induction inhibits
%  the assembly of chemoreceptor signaling clusters in salmonella enterica.
%\newblock {\em PLoS One} 11(1):e0146685.
%
%\bibitem{Logan2018}
%Logan SL, et~al. (2018) {The Vibrio cholerae type VI secretion system can
%  modulate host intestinal mechanics to displace gut bacterial symbionts}.
%\newblock {\em Proceedings of the National Academy of Sciences}
%  115(16):E3779--E3787.
%
%\bibitem{ganz2018}
%Ganz J, et~al. (2018) Image velocimetry and spectral analysis enable
%  quantitative characterization of larval zebrafish gut motility.
%\newblock {\em Neurogastroenterology \& Motility} 30(9):e13351.
%
%\bibitem{robinson_experimental_2018}
%Robinson CD, et~al. (2018) Experimental bacterial adaptation to the zebrafish
%  gut reveals a primary role for immigration.
%\newblock {\em PLoS biology} 16(12):e2006893.
%
%\bibitem{krapivsky2010kinetic}
%Krapivsky PL, Redner S, Ben-Naim E (2010) {\em A Kinetic View of Statistical
%  Physics}.
%\newblock (Cambridge University Press).
%
%\bibitem{krapivsky1996transitional}
%Krapivsky P, Redner S (1996) Transitional aggregation kinetics in dry and damp
%  environments.
%\newblock {\em Physical Review E} 54(4):3553.
%
%\bibitem{bansept2019enchained}
%Bansept F, et~al. (2019) Enchained growth and cluster dislocation: a possible
%  mechanism for microbiota homeostasis.
%\newblock {\em bioRxiv}, doi: 10.1101/298059, 2019.
%
%\bibitem{baharoglu2011vibrio}
%Baharoglu Z, Mazel D (2011) Vibrio cholerae triggers sos and mutagenesis in
%  response to a wide range of antibiotics: a route towards multiresistance.
%\newblock {\em Antimicrobial Agents and Chemotherapy} 55(5):2438--2441.
%
%\bibitem{guerin2009sos}
%Guerin {\'E}, et~al. (2009) The sos response controls integron recombination.
%\newblock {\em Science} 324(5930):1034--1034.
%
%\bibitem{beaber2004sos}
%Beaber JW, Hochhut B, Waldor MK (2004) Sos response promotes horizontal
%  dissemination of antibiotic resistance genes.
%\newblock {\em Nature} 427(6969):72.
%
%\bibitem{tropini_gut_2017}
%Tropini C, Earle KA, Huang KC, Sonnenburg JL (2017) The gut microbiome:
%  connecting spatial organization to function.
%\newblock {\em Cell Host \& Microbe} 21(4):433--442.
%
%\bibitem{westerfield_zebrafish_2007}
%Westerfield M (2007) {The Zebrafish Book. A Guide for the Laboratory Use of
%  Zebrafish (Danio rerio), 5th Edition}.
%\newblock {\em University of Oregon Press, Eugene (Book)}.
%
%\bibitem{melancon_best_2017}
%Melancon E, et~al. (2017) {Best practices for germ-free derivation and
%  gnotobiotic zebrafish husbandry}.
%\newblock {\em Methods in Cell Biology} 138:61--100.
%
%\bibitem{milligan-myhre_study_2011}
%Milligan-Myhre K, et~al. (2011) Study of host--microbe interactions in
%  zebrafish in {\em Methods in Cell Biology}.
%\newblock (Elsevier) Vol.{} 105, pp. 87--116.
%
%\bibitem{jemielita_spatial_2014}
%Jemielita M, et~al. (2014) Spatial and temporal features of the growth of a
%  bacterial species colonizing the zebrafish gut.
%\newblock {\em mBio} 5(6):1--8.
%
%\bibitem{netuschil2014confusion}
%Netuschil L, Auschill TM, Sculean A, Arweiler NB (2014) Confusion over
%  live/dead stainings for the detection of vital microorganisms in oral
%  biofilms-which stain is suitable?
%\newblock {\em BMC Oral Health} 14(1):2.
%
%\bibitem{gillespie1977exact}
%Gillespie DT (1977) Exact stochastic simulation of coupled chemical reactions.
%\newblock {\em The Journal of Physical Chemistry} 81(25):2340--2361.
%
%\bibitem{gillespie2001approximate}
%Gillespie DT (2001) Approximate accelerated stochastic simulation of chemically
%  reacting systems.
%\newblock {\em The Journal of Chemical Physics} 115(4):1716--1733.
%
%\end{thebibliography}


